{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Write out Context for RAFT\n",
    "This notebook quickly goes over making a context .txt file to pass to the RAFT dataset construction script `raft-dataset.py`.  The `fetch_context` function can be found in `genraitor/rag/uniprot_api.py`."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "from genraitor.rag.uniprot_api import fetch_context"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "2024-08-14 22:43:22 a100-05 root[75961] INFO Retrieval context for uniprot id P05067\n",
      "2024-08-14 22:43:22 a100-05 root[75961] INFO Retrieval context for uniprot id Q9NQA5\n",
      "2024-08-14 22:43:22 a100-05 root[75961] INFO Retrieval context for uniprot id Q15582\n",
      "2024-08-14 22:43:43 a100-05 root[75961] INFO Retrieval context for uniprot id P61927\n",
      "2024-08-14 22:43:48 a100-05 root[75961] INFO Retrieval context for uniprot id P49746\n",
      "2024-08-14 22:43:50 a100-05 root[75961] INFO Retrieval context for uniprot id P35442\n",
      "2024-08-14 22:43:56 a100-05 root[75961] INFO Retrieval context for uniprot id P07477\n",
      "2024-08-14 22:44:08 a100-05 root[75961] INFO Retrieval context for uniprot id P35443\n",
      "2024-08-14 22:44:13 a100-05 root[75961] INFO Retrieval context for uniprot id Q9P035\n"
     ]
    }
   ],
   "source": [
    "# Fetch context for some uniprot Accession ID's\n",
    "# These MUST be the Accession ID's, not the other uniprot identifiers.\n",
    "abstracts, interactions, pathways = fetch_context(\n",
    "    ['P05067', \n",
    "    'Q9NQA5',\n",
    "    'Q15582',\n",
    "    'P61927',\n",
    "    'P49746',\n",
    "    'P35442',\n",
    "    'P07477',\n",
    "    'P35443',\n",
    "    'Q9P035']\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Custom construction of the context\n",
    "contexts = []\n",
    "\n",
    "for a, i, p in zip(abstracts, interactions, pathways):\n",
    "    cur_abstracts = \"ABSTRACTS:\\n\" + \"\\n\".join(a) + \"\\n\"\n",
    "    cur_interactions = \"INTERACTIONS:\\n\" + \"\\n\".join(i) + \"\\n\"\n",
    "    cur_pathways = \"PATHWAYS:\\n\" + \"\\n\".join(p) + \"\\n\"\n",
    "\n",
    "    contexts.append(cur_abstracts + cur_interactions + cur_pathways)\n",
    "    \n",
    "contexts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "full_context = \"<BEGIN_CONTEXT>\" + \"<END_CONTEXT>\\n\\n<BEGIN_CONTEXT>\".join(contexts) + \"<END_CONTEXT>\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<BEGIN_CONTEXT>ABSTRACTS:\n",
      "Alzheimer's disease is characterized by a widespread functional disturbance of the human brain. Fibrillar amyloid proteins are deposited inside neurons as neurofibrillary tangles and extracellularly as amyloid plaque cores and in blood vessels. The major protein subunit (A4) of the amyloid fibril of tangles, plaques and blood vessel deposits is an insoluble, highly aggregating small polypeptide of relative molecular mass 4,500. The same polypeptide is also deposited in the brains of aged individuals with trisomy 21 (Down's syndrome). We have argued previously that the A4 protein is of neuronal origin and is the cleavage product of a larger precursor protein. To identify this precursor, we have now isolated and sequenced an apparently full-length complementary DNA clone coding for the A4 polypeptide. The predicted precursor consists of 695 residues and contains features characteristic of glycosylated cell-surface receptors. This sequence, together with the localization of its gene on chromosome 21, suggests that the cerebral amyloid deposited in Alzheimer's disease and aged Down's syndrome is caused by aberrant catabolism of a cell-surface receptor.\n",
      "The amyloid proteins isolated from neuritic plaques and the cerebrovasculature of Alzheimer's disease are self-aggregating moieties termed A4 protein and beta-protein, respectively. A putative A4 amyloid precursor (herein termed A4(695] has been characterized by analysis of a human brain complementary DNA. We report here the sequence of a closely related amyloid cDNA, A4(751), distinguished from A4(695) by the presence of a 168 base-pair (bp) sequence which adds 57 amino acids to, and removes one residue from, the predicted A4(695) protein. The peptide predicted from this insert is very similar to the Kunitz family of serine proteinase inhibitors. The two A4-specific messenger RNAs are differentially expressed: in a limited survey, A4(751) mRNA appears to be ubiquitous, whereas A4(695) mRNA has a restricted pattern of expression which includes cells from neuronal tissue. These data may have significant implications for understanding amyloid deposition in Alzheimer's disease.\n",
      "Alzheimer's disease (AD) is characterized by the cerebral deposition of fibrillar aggregates of the amyloid A4 protein. Complementary DNA's coding for the precursor of the amyloid A4 protein have been described. In order to identify the structure of the precursor gene relevant clones from several human genomic libraries were isolated. Sequence analysis of the various clones revealed 16 exons to encode the 695 residue precursor protein (PreA4(695] of Alzheimer's disease amyloid A4 protein. The DNA sequence coding for the amyloid A4 protein is interrupted by an intron. This finding supports the idea that amyloid A4 protein arises by incomplete proteolysis of a larger precursor, and not by aberrant splicing.\n",
      "Amyloid beta-protein (BP) deposited in Alzheimer brains is a cleavage product of a larger precursor (BPP). The BPP gene encodes three types of mRNA generated by alternative splicing, two of which contain the sequence encoding Kunitz-type serine-protease inhibitor (serpin). To investigate the regulatory mechanisms of BPP synthesis at the gene level, we isolated 36 genomic DNA clones covering all the exons of the human BPP gene. This gene consists of 18 exons and spans more than 170 kb. BP is encoded by the 16th and the 17th exons and the serpin domain by the 7th exon. Sequence analysis showed that the 7th and 8th introns lack a typical branchpoint for splicing. This might relate to the alternative splicing. The promoter of the BPP gene has some characteristics of those of housekeeping genes and contains a number of possible methylation sites. The methylation status of the promoter was analyzed by Southern blotting but no alteration was observed among tissues and between control and Alzheimer brains. We also tested the roles of two possible activator protein-1-binding sites and a possible heat-shock element found within the promoter. Northern blotting showed that the transcription of the BPP gene was apparently induced by 12-O-tetradecanoylphorbol-13-acetate (phorbol derivative) in HeLa cells.\n",
      "The gene for the beta A4-amyloid precursor protein (APP) consists of 19 exons which code for a typical N- and O-glycosylated transmembrane protein with four extracellular domains followed by the transmembrane domain and a short cytoplasmic domain. The beta A4-amyloid sequence is part of exons 16 and 17. Several APP isoforms can be generated by alternative splicing of exons 7 and 8, encoding domains with homologies to Kunitz-type protease inhibitors and the MRC OX-2 antigen, respectively. The mechanism by which the pathological beta A4 is generated is unknown, it is however a critical event in Alzheimer's disease and is distinct from the normally occurring cleavage and secretion of APPs within the beta A4 sequence. We report here for the first time considerable APP mRNA expression by rat brain microglial cells. In addition we showed by S1 nuclease protection and polymerase chain reaction analysis of reverse transcribed RNA (RT-PCR) that T-lymphocytes, macrophages, and microglial cells expressed a new APP isoform by selection of a novel alternative splice site and exclusion of exon 15 of the APP gene. This leads to a transmembrane, beta A4 sequence containing APP variant, lacking 18 amino acid residues close to the amyloidogenic region. The use of this novel alternative splice site alters the structure of APP in close proximity to the beta A4 region and thus may determine a variant, potentially pathogenic processing of leukocyte-derived APP in brain.\n",
      "We developed a novel in vitro method for making nested deletions and applied it to a large-scale DNA sequencing. A DNA fragment to be sequenced (up to 15 kb long) was cloned with a new vector possessing two unique Sfi I sites, digested by Sfi I and ligated to generate a large head-to-tail concatemer. The large concatemer was randomly fragmented by sonication and then redigested by Sfi I to separate insert and vector DNAs. The fragments of various length were then cloned into the other vector(s) specifically designed for selective cloning of insert-derived DNA fragments to generate a library of nested deletions. This method allowed a single person to generate >20 nested deletion libraries sufficient to cover 100 kb in a few days. We applied the method for sequencing of P1 clones and successfully determined the complete sequence of approximately 300 kb of the human amyloid precursor protein (APP) locus on chromosome 21 with a redundancy of 3.8, reasonably low cost and very few gaps remaining to be closed. Development of some new instruments and software is also described which makes this method more applicable for large-scale sequencing.\n",
      "Senile plaques, neurofibrillary tangles and amyloid-laden cerebral vessels are characteristic features in the brains of individuals with Alzheimer's disease (AD). The principal component of amyloid in senile plaque and amyloid-laden cerebral vessels is beta-amyloid (Abeta), a peptide proteolytically derived from a large amyloid precursor protein (APP). To date, several alternatively spliced human APP transcripts have been described. Here, we report the identification of a novel alternative splicing isoform of the human APP gene, APP639, which excludes exon 2 as well as exons 7 and 8. RT-PCR and Southern blot analysis show that APP639 mRNA is expressed in many human fetal tissues. In contrast, the APP639 transcript is hardly detected in the aged human cerebral cortex from both pathologically confirmed sporadic AD cases and nondemented controls. However, APP639 mRNA exists in the adult human liver. Western blot analysis shows that the protein product produced from the APP639 cDNA could be recognized by the APP antibody, and it does lack the exon 2 coding region. These results suggest that APP639, a novel alternative splicing isoform of human APP gene, does exist in human tissues in vivo.\n",
      "As a base for human transcriptome and functional genomics, we created the \"full-length long Japan\" (FLJ) collection of sequenced human cDNAs. We determined the entire sequence of 21,243 selected clones and found that 14,490 cDNAs (10,897 clusters) were unique to the FLJ collection. About half of them (5,416) seemed to be protein-coding. Of those, 1,999 clusters had not been predicted by computational methods. The distribution of GC content of nonpredicted cDNAs had a peak at approximately 58% compared with a peak at approximately 42%for predicted cDNAs. Thus, there seems to be a slight bias against GC-rich transcripts in current gene prediction procedures. The rest of the cDNAs unique to the FLJ collection (5,481) contained no obvious open reading frames (ORFs) and thus are candidate noncoding RNAs. About one-fourth of them (1,378) showed a clear pattern of splicing. The distribution of GC content of noncoding cDNAs was narrow and had a peak at approximately 42%, relatively low compared with that of protein-coding cDNAs.\n",
      "Chromosome 21 is the smallest human autosome. An extra copy of chromosome 21 causes Down syndrome, the most frequent genetic cause of significant mental retardation, which affects up to 1 in 700 live births. Several anonymous loci for monogenic disorders and predispositions for common complex disorders have also been mapped to this chromosome, and loss of heterozygosity has been observed in regions associated with solid tumours. Here we report the sequence and gene catalogue of the long arm of chromosome 21. We have sequenced 33,546,361 base pairs (bp) of DNA with very high accuracy, the largest contig being 25,491,867 bp. Only three small clone gaps and seven sequencing gaps remain, comprising about 100 kilobases. Thus, we achieved 99.7% coverage of 21q. We also sequenced 281,116 bp from the short arm. The structural features identified include duplications that are probably involved in chromosomal abnormalities and repeat structures in the telomeric and pericentromeric regions. Analysis of the chromosome revealed 127 known genes, 98 predicted genes and 59 pseudogenes.\n",
      "The National Institutes of Health's Mammalian Gene Collection (MGC) project was designed to generate and sequence a publicly accessible cDNA resource containing a complete open reading frame (ORF) for every human and mouse gene. The project initially used a random strategy to select clones from a large number of cDNA libraries from diverse tissues. Candidate clones were chosen based on 5'-EST sequences, and then fully sequenced to high accuracy and analyzed by algorithms developed for this project. Currently, more than 11,000 human and 10,000 mouse genes are represented in MGC by at least one clone with a full ORF. The random selection approach is now reaching a saturation point, and a transition to protocols targeted at the missing transcripts is now required to complete the mouse and human collections. Comparison of the sequence of the MGC clones to reference genome sequences reveals that most cDNA clones are of very high sequence quality, although it is likely that some cDNAs may carry missense variants as a consequence of experimental artifact, such as PCR, cloning, or reverse transcriptase errors. Recently, a rat cDNA component was added to the project, and ongoing frog (Xenopus) and zebrafish (Danio) cDNA projects were expanded to take advantage of the high-throughput MGC pipeline.\n",
      "Human genomic clones encoding the promoter region and the first two exons of the beta-amyloid protein precursor (beta-APP) gene were isolated. The first exon is 205 base pairs (bp) long and encodes 19 amino acids. The second exon is 168 bp long and encodes 56 amino acids. The 5'-flanking sequence of the beta-APP gene was found to display promoter activity in several cell lines including PC12 cells where the highest activity was detected. The promoter region of this gene lacks the typical \"TATAA\" and \"CAAT\" boxes usually associated with eukaryotic promoters. Five copies of the GGGCGC sequence are located between positions -107 and -188 and one copy is located within the first exon of the beta-APP gene. Consensus sequences recognized by the transcription factors Sp1 and AP-1 are located upstream from the RNA start site. Palindromic sequences capable of forming stable hairpin-like structures are found around the main transcription initiation site. The structural characteristics of the beta-APP promoter indicate that multiple elements participate in the regulation of the expression of this gene.\n",
      "Normal human fibroblasts secrete a protein named protease nexin II (PN II) which previously was shown to form sodium dodecyl sulfate (SDS)-stable complexes with epidermal growth factor-binding protein (EGF-BP). These complexes then bind to the same cells and are rapidly internalized and degraded (Knauer, D.J., and Cunningham, D.D. (1982) Proc. Natl. Acad. Sci. U.S.A. 79, 2310-2314). Here we describe a procedure for purifying PN II to apparent homogeneity from serum-free culture medium conditioned by human fibroblasts. The first step employed dextran sulfate-Sepharose affinity chromatography. Further purification was achieved by ion-exchange chromatography on DEAE-Sepharose followed by gel filtration on Sephacryl S-400. Sequence analysis of purified PN II identified 33 amino-terminal amino acids; a computer search of several protein sequence data banks failed to reveal homologies with other reported amino acid sequences. Purified PN II had an apparent Mr of 106,000 and an isoelectric point of approximately 7.2. It retained full activity after incubation in the presence of 0.05% SDS or at a pH of 1.5. PN II formed SDS-stable complexes with EGF-BP, the gamma subunit of 7 S nerve growth factor, and trypsin with estimated Mr of 120,000, 120,000, and 110,000, respectively. PN II was metabolically labeled with [35S]methionine and purified; the metabolically labeled protein formed complexes with EGF-BP. Complexes between purified PN II and EGF-BP bound to human fibroblasts. These results show that the purified protein possesses the properties previously attributed to PN II in cell culture medium.\n",
      "Current non-gel techniques for analyzing proteomes rely heavily on mass spectrometric analysis of enzymatically digested protein mixtures. Prior to analysis, a highly complex peptide mixture is either separated on a multidimensional chromatographic system or it is first reduced in complexity by isolating sets of representative peptides. Recently, we developed a peptide isolation procedure based on diagonal electrophoresis and diagonal chromatography. We call it combined fractional diagonal chromatography (COFRADIC). In previous experiments, we used COFRADIC to identify more than 800 Escherichia coli proteins by tandem mass spectrometric (MS/MS) analysis of isolated methionine-containing peptides. Here, we describe a diagonal method to isolate N-terminal peptides. This reduces the complexity of the peptide sample, because each protein has one N terminus and is thus represented by only one peptide. In this new procedure, free amino groups in proteins are first blocked by acetylation and then digested with trypsin. After reverse-phase (RP) chromatographic fractionation of the generated peptide mixture, internal peptides are blocked using 2,4,6-trinitrobenzenesulfonic acid (TNBS); they display a strong hydrophobic shift and therefore segregate from the unaltered N-terminal peptides during a second identical separation step. N-terminal peptides can thereby be specifically collected for further liquid chromatography (LC)-MS/MS analysis. Omitting the acetylation step results in the isolation of non-lysine-containing N-terminal peptides from in vivo blocked proteins.\n",
      "INTERACTIONS:\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9NY61\", \"geneName\": \"AATF\", \"intActId\": \"EBI-372428\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P16112\", \"geneName\": \"ACAN\", \"intActId\": \"EBI-9076211\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P60709\", \"geneName\": \"ACTB\", \"intActId\": \"EBI-353944\"}, \"numberOfExperiments\": 8, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P61158\", \"geneName\": \"ACTR3\", \"intActId\": \"EBI-351428\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O14672\", \"geneName\": \"ADAM10\", \"intActId\": \"EBI-1536151\"}, \"numberOfExperiments\": 7, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"A0AVL1\", \"geneName\": \"ADAM9\", \"intActId\": \"EBI-25935864\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P18509\", \"geneName\": \"ADCYAP1\", \"intActId\": \"EBI-8588930\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P41586-2\", \"geneName\": \"ADCYAP1R1\", \"intActId\": \"EBI-17241711\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q15109\", \"geneName\": \"AGER\", \"intActId\": \"EBI-1646426\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q13155\", \"geneName\": \"AIMP2\", \"intActId\": \"EBI-745226\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P63010-2\", \"geneName\": \"AP2B1\", \"intActId\": \"EBI-11529439\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q02410\", \"geneName\": \"APBA1\", \"intActId\": \"EBI-368690\"}, \"numberOfExperiments\": 5, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q99767\", \"geneName\": \"APBA2\", \"intActId\": \"EBI-81711\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O96018\", \"geneName\": \"APBA3\", \"intActId\": \"EBI-6115839\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O00213\", \"geneName\": \"APBB1\", \"intActId\": \"EBI-81694\"}, \"numberOfExperiments\": 8, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O00213-2\", \"geneName\": \"APBB1\", \"intActId\": \"EBI-13307975\"}, \"numberOfExperiments\": 6, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q92870\", \"geneName\": \"APBB2\", \"intActId\": \"EBI-79277\"}, \"numberOfExperiments\": 4, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q92870-2\", \"geneName\": \"APBB2\", \"intActId\": \"EBI-21535880\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O95704\", \"geneName\": \"APBB3\", \"intActId\": \"EBI-286427\"}, \"numberOfExperiments\": 6, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P02743\", \"geneName\": \"APCS\", \"intActId\": \"EBI-2115799\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q96BI3\", \"geneName\": \"APH1A\", \"intActId\": \"EBI-2606935\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8WW43\", \"geneName\": \"APH1B\", \"intActId\": \"EBI-2606497\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q06481-5\", \"geneName\": \"APLP2\", \"intActId\": \"EBI-25646567\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P02647\", \"geneName\": \"APOA1\", \"intActId\": \"EBI-701692\"}, \"numberOfExperiments\": 8, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P05067\", \"geneName\": \"APP\", \"intActId\": \"EBI-77613\"}, \"numberOfExperiments\": 107, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q92624\", \"geneName\": \"APPBP2\", \"intActId\": \"EBI-743771\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q6P4J0\", \"geneName\": \"ARD1A\", \"intActId\": \"EBI-10252815\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P61204\", \"geneName\": \"ARF3\", \"intActId\": \"EBI-641535\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q0P5N6\", \"geneName\": \"ARL16\", \"intActId\": \"EBI-10186132\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P56211\", \"geneName\": \"ARPP19\", \"intActId\": \"EBI-5773880\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P05026\", \"geneName\": \"ATP1B1\", \"intActId\": \"EBI-714630\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P54253\", \"geneName\": \"ATXN1\", \"intActId\": \"EBI-930964\"}, \"numberOfExperiments\": 8, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P56817\", \"geneName\": \"BACE1\", \"intActId\": \"EBI-2433139\"}, \"numberOfExperiments\": 11, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9Y5Z0\", \"geneName\": \"BACE2\", \"intActId\": \"EBI-11282723\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q92934\", \"geneName\": \"BAD\", \"intActId\": \"EBI-700771\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P46379-2\", \"geneName\": \"BAG6\", \"intActId\": \"EBI-10988864\"}, \"numberOfExperiments\": 5, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q96GW7\", \"geneName\": \"BCAN\", \"intActId\": \"EBI-2690445\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P51572\", \"geneName\": \"BCAP31\", \"intActId\": \"EBI-77683\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P10415\", \"geneName\": \"BCL2\", \"intActId\": \"EBI-77694\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P23560-2\", \"geneName\": \"BDNF\", \"intActId\": \"EBI-12275524\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O15392\", \"geneName\": \"BIRC5\", \"intActId\": \"EBI-518823\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q13867\", \"geneName\": \"BLMH\", \"intActId\": \"EBI-718504\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P35613\", \"geneName\": \"BSG\", \"intActId\": \"EBI-750709\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8IU99\", \"geneName\": \"CALHM1\", \"intActId\": \"EBI-1790341\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P0DP25\", \"geneName\": \"CALM3\", \"intActId\": \"EBI-397435\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P27797\", \"geneName\": \"CALR\", \"intActId\": \"EBI-1049597\"}, \"numberOfExperiments\": 5, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O43852-3\", \"geneName\": \"CALU\", \"intActId\": \"EBI-11536607\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9UQM7\", \"geneName\": \"CAMK2A\", \"intActId\": \"EBI-1383687\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P27824-2\", \"geneName\": \"CANX\", \"intActId\": \"EBI-25890990\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P07384\", \"geneName\": \"CAPN1\", \"intActId\": \"EBI-1542113\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P29466-3\", \"geneName\": \"CASP1\", \"intActId\": \"EBI-12248206\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P42574\", \"geneName\": \"CASP3\", \"intActId\": \"EBI-524064\"}, \"numberOfExperiments\": 4, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q14790\", \"geneName\": \"CASP8\", \"intActId\": \"EBI-78060\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q03135\", \"geneName\": \"CAV1\", \"intActId\": \"EBI-603614\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P83916\", \"geneName\": \"CBX1\", \"intActId\": \"EBI-78129\"}, \"numberOfExperiments\": 6, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P40227\", \"geneName\": \"CCT6A\", \"intActId\": \"EBI-356687\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P16671\", \"geneName\": \"CD36\", \"intActId\": \"EBI-2808214\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q08722-3\", \"geneName\": \"CD47\", \"intActId\": \"EBI-17263290\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P06493\", \"geneName\": \"CDK1\", \"intActId\": \"EBI-444308\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q00535\", \"geneName\": \"CDK5\", \"intActId\": \"EBI-1041567\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P42773\", \"geneName\": \"CDKN2C\", \"intActId\": \"EBI-711290\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P43681\", \"geneName\": \"CHRNA4\", \"intActId\": \"EBI-7132379\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P36544\", \"geneName\": \"CHRNA7\", \"intActId\": \"EBI-79333\"}, \"numberOfExperiments\": 4, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q16740\", \"geneName\": \"CLPP\", \"intActId\": \"EBI-1056029\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O94985-2\", \"geneName\": \"CLSTN1\", \"intActId\": \"EBI-16041593\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8IUW6\", \"geneName\": \"CLSTN3\", \"intActId\": \"EBI-25832219\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P10909\", \"geneName\": \"CLU\", \"intActId\": \"EBI-1104674\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P26441\", \"geneName\": \"CNTF\", \"intActId\": \"EBI-1050897\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q02246\", \"geneName\": \"CNTN2\", \"intActId\": \"EBI-4397248\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8NE08\", \"geneName\": \"COL25A1\", \"intActId\": \"EBI-25836642\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q96A83-2\", \"geneName\": \"COL26A1\", \"intActId\": \"EBI-21553822\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P29400-2\", \"geneName\": \"COL4A5\", \"intActId\": \"EBI-12211159\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q14031\", \"geneName\": \"COL4A6\", \"intActId\": \"EBI-2432407\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P31146\", \"geneName\": \"CORO1A\", \"intActId\": \"EBI-1046676\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P20674\", \"geneName\": \"COX5A\", \"intActId\": \"EBI-715032\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P15086\", \"geneName\": \"CPB1\", \"intActId\": \"EBI-25936844\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P02511\", \"geneName\": \"CRYAB\", \"intActId\": \"EBI-739060\"}, \"numberOfExperiments\": 7, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P48730\", \"geneName\": \"CSNK1D\", \"intActId\": \"EBI-751621\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P48730-2\", \"geneName\": \"CSNK1D\", \"intActId\": \"EBI-9087876\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P68400\", \"geneName\": \"CSNK2A1\", \"intActId\": \"EBI-347804\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P01034\", \"geneName\": \"CST3\", \"intActId\": \"EBI-948622\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P49711\", \"geneName\": \"CTCF\", \"intActId\": \"EBI-932887\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P07339\", \"geneName\": \"CTSD\", \"intActId\": \"EBI-2115097\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P99999\", \"geneName\": \"CYCS\", \"intActId\": \"EBI-446479\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O75553-4\", \"geneName\": \"DAB1\", \"intActId\": \"EBI-21246842\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P98082\", \"geneName\": \"DAB2\", \"intActId\": \"EBI-1171238\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q14203-5\", \"geneName\": \"DCTN1\", \"intActId\": \"EBI-25840379\"}, \"numberOfExperiments\": 5, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q13561\", \"geneName\": \"DCTN2\", \"intActId\": \"EBI-715074\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q6I9W9\", \"geneName\": \"DKFZP586N0721\", \"intActId\": \"EBI-25927172\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O14645\", \"geneName\": \"DNALI1\", \"intActId\": \"EBI-395638\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q01658\", \"geneName\": \"DR1\", \"intActId\": \"EBI-750300\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P21917\", \"geneName\": \"DRD4\", \"intActId\": \"EBI-8592297\"}, \"numberOfExperiments\": 6, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q16828\", \"geneName\": \"DUSP6\", \"intActId\": \"EBI-746870\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q92997\", \"geneName\": \"DVL3\", \"intActId\": \"EBI-739789\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O14576-2\", \"geneName\": \"DYNC1I1\", \"intActId\": \"EBI-25840445\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O14576-5\", \"geneName\": \"DYNC1I1\", \"intActId\": \"EBI-25936079\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q01094\", \"geneName\": \"E2F1\", \"intActId\": \"EBI-448924\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q3B7T1\", \"geneName\": \"EDRF1\", \"intActId\": \"EBI-2870947\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P20042\", \"geneName\": \"EIF2S2\", \"intActId\": \"EBI-711977\"}, \"numberOfExperiments\": 6, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P19419\", \"geneName\": \"ELK1\", \"intActId\": \"EBI-726632\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P11171-2\", \"geneName\": \"EPB41\", \"intActId\": \"EBI-10197451\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P11171-7\", \"geneName\": \"EPB41\", \"intActId\": \"EBI-25852354\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9BS26\", \"geneName\": \"ERP44\", \"intActId\": \"EBI-541644\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P00748\", \"geneName\": \"F12\", \"intActId\": \"EBI-6378830\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P00734\", \"geneName\": \"F2\", \"intActId\": \"EBI-297094\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P23142-4\", \"geneName\": \"FBLN1\", \"intActId\": \"EBI-11956479\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q92915\", \"geneName\": \"FGF14\", \"intActId\": \"EBI-10489272\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P62942\", \"geneName\": \"FKBP1A\", \"intActId\": \"EBI-1027571\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P21333-2\", \"geneName\": \"FLNA\", \"intActId\": \"EBI-9641086\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O75955\", \"geneName\": \"FLOT1\", \"intActId\": \"EBI-603643\"}, \"numberOfExperiments\": 5, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9BTI6\", \"geneName\": \"FLOT2\", \"intActId\": \"EBI-23703366\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P01100\", \"geneName\": \"FOS\", \"intActId\": \"EBI-852851\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P25090\", \"geneName\": \"FPR2\", \"intActId\": \"EBI-17291771\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P09958\", \"geneName\": \"FURIN\", \"intActId\": \"EBI-1056807\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P06241\", \"geneName\": \"FYN\", \"intActId\": \"EBI-515315\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P04406\", \"geneName\": \"GAPDH\", \"intActId\": \"EBI-354056\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9UJY5-4\", \"geneName\": \"GGA1\", \"intActId\": \"EBI-12108696\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q05586\", \"geneName\": \"GRIN1\", \"intActId\": \"EBI-998542\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P25098\", \"geneName\": \"GRK2\", \"intActId\": \"EBI-3904795\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P43250\", \"geneName\": \"GRK6\", \"intActId\": \"EBI-722747\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P43250-2\", \"geneName\": \"GRK6\", \"intActId\": \"EBI-6428342\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"A4D1B5\", \"geneName\": \"GSAP\", \"intActId\": \"EBI-15875313\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P49841-2\", \"geneName\": \"GSK3B\", \"intActId\": \"EBI-15870655\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q03013\", \"geneName\": \"GSTM4\", \"intActId\": \"EBI-713363\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q00403\", \"geneName\": \"GTF2B\", \"intActId\": \"EBI-389564\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9Y5Q9\", \"geneName\": \"GTF3C3\", \"intActId\": \"EBI-1054873\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P09429\", \"geneName\": \"HMGB1\", \"intActId\": \"EBI-389432\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P30519\", \"geneName\": \"HMOX2\", \"intActId\": \"EBI-712096\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9UJC3\", \"geneName\": \"HOOK1\", \"intActId\": \"EBI-746704\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q99714\", \"geneName\": \"HSD17B10\", \"intActId\": \"EBI-79964\"}, \"numberOfExperiments\": 7, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q99714-2\", \"geneName\": \"HSD17B10\", \"intActId\": \"EBI-25939412\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P07900\", \"geneName\": \"HSP90AA1\", \"intActId\": \"EBI-296047\"}, \"numberOfExperiments\": 5, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P14625\", \"geneName\": \"HSP90B1\", \"intActId\": \"EBI-359129\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P11021\", \"geneName\": \"HSPA5\", \"intActId\": \"EBI-354921\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P11142\", \"geneName\": \"HSPA8\", \"intActId\": \"EBI-351896\"}, \"numberOfExperiments\": 8, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P04792\", \"geneName\": \"HSPB1\", \"intActId\": \"EBI-352682\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q16082\", \"geneName\": \"HSPB2\", \"intActId\": \"EBI-739395\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P10809\", \"geneName\": \"HSPD1\", \"intActId\": \"EBI-352528\"}, \"numberOfExperiments\": 6, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P42858\", \"geneName\": \"HTT\", \"intActId\": \"EBI-466029\"}, \"numberOfExperiments\": 6, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9UMF0\", \"geneName\": \"ICAM5\", \"intActId\": \"EBI-6398041\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P14735\", \"geneName\": \"IDE\", \"intActId\": \"EBI-2556886\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q16352\", \"geneName\": \"INA\", \"intActId\": \"EBI-366258\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q6DN90-2\", \"geneName\": \"IQSEC1\", \"intActId\": \"EBI-21911304\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P05556\", \"geneName\": \"ITGB1\", \"intActId\": \"EBI-703066\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9Y287\", \"geneName\": \"ITM2B\", \"intActId\": \"EBI-2866431\"}, \"numberOfExperiments\": 6, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P05412\", \"geneName\": \"JUN\", \"intActId\": \"EBI-852823\"}, \"numberOfExperiments\": 5, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P17535\", \"geneName\": \"JUND\", \"intActId\": \"EBI-2682803\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q92993\", \"geneName\": \"KAT5\", \"intActId\": \"EBI-399080\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q92993-2\", \"geneName\": \"KAT5\", \"intActId\": \"EBI-20795332\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q13303\", \"geneName\": \"KCNAB2\", \"intActId\": \"EBI-948729\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9Y2W7\", \"geneName\": \"KCNIP3\", \"intActId\": \"EBI-751501\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O60333-2\", \"geneName\": \"KIF1B\", \"intActId\": \"EBI-10975473\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q07866-2\", \"geneName\": \"KLC1\", \"intActId\": \"EBI-11979975\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O14901\", \"geneName\": \"KLF11\", \"intActId\": \"EBI-948266\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q92876\", \"geneName\": \"KLK6\", \"intActId\": \"EBI-2432309\"}, \"numberOfExperiments\": 4, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P01116-2\", \"geneName\": \"KRAS\", \"intActId\": \"EBI-367427\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q16363-3\", \"geneName\": \"LAMA4\", \"intActId\": \"EBI-17719490\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9BYZ2\", \"geneName\": \"LDHAL6B\", \"intActId\": \"EBI-1108377\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q96FE5\", \"geneName\": \"LINGO1\", \"intActId\": \"EBI-719955\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q07954-2\", \"geneName\": \"LRP1\", \"intActId\": \"EBI-25833471\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9NZR2\", \"geneName\": \"LRP1B\", \"intActId\": \"EBI-1642131\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P30533\", \"geneName\": \"LRPAP1\", \"intActId\": \"EBI-715927\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P42704\", \"geneName\": \"LRPPRC\", \"intActId\": \"EBI-1050853\"}, \"numberOfExperiments\": 8, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P07948\", \"geneName\": \"LYN\", \"intActId\": \"EBI-79452\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9GZQ8\", \"geneName\": \"MAP1LC3B\", \"intActId\": \"EBI-373144\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P36507\", \"geneName\": \"MAP2K2\", \"intActId\": \"EBI-1056930\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P28482\", \"geneName\": \"MAPK1\", \"intActId\": \"EBI-959949\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P53778\", \"geneName\": \"MAPK12\", \"intActId\": \"EBI-602406\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9UQF2\", \"geneName\": \"MAPK8IP1\", \"intActId\": \"EBI-78404\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P10636\", \"geneName\": \"MAPT\", \"intActId\": \"EBI-366182\"}, \"numberOfExperiments\": 8, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P10636-8\", \"geneName\": \"MAPT\", \"intActId\": \"EBI-366233\"}, \"numberOfExperiments\": 4, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9P0L2\", \"geneName\": \"MARK1\", \"intActId\": \"EBI-968587\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q6IPE9\", \"geneName\": \"MARK4\", \"intActId\": \"EBI-10250211\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q96L34\", \"geneName\": \"MARK4\", \"intActId\": \"EBI-302319\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q00266\", \"geneName\": \"MAT1A\", \"intActId\": \"EBI-967087\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P02686-2\", \"geneName\": \"MBP\", \"intActId\": \"EBI-12159027\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q93074\", \"geneName\": \"MED12\", \"intActId\": \"EBI-394357\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8TDB4\", \"geneName\": \"MGARP\", \"intActId\": \"EBI-4397720\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O94851\", \"geneName\": \"MICAL2\", \"intActId\": \"EBI-2804835\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"A4FUJ8\", \"geneName\": \"MKL1\", \"intActId\": \"EBI-21250407\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P08473\", \"geneName\": \"MME\", \"intActId\": \"EBI-353759\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P08253\", \"geneName\": \"MMP2\", \"intActId\": \"EBI-1033518\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q99547\", \"geneName\": \"MPHOSPH6\", \"intActId\": \"EBI-373187\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8N594\", \"geneName\": \"MPND\", \"intActId\": \"EBI-2512452\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P41227\", \"geneName\": \"NAA10\", \"intActId\": \"EBI-747693\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q13765\", \"geneName\": \"NACA\", \"intActId\": \"EBI-712216\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q13564\", \"geneName\": \"NAE1\", \"intActId\": \"EBI-718631\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P41271-2\", \"geneName\": \"NBL1\", \"intActId\": \"EBI-12135485\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P19404\", \"geneName\": \"NDUFV2\", \"intActId\": \"EBI-713665\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O76041\", \"geneName\": \"NEBL\", \"intActId\": \"EBI-2880203\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P12036\", \"geneName\": \"NEFH\", \"intActId\": \"EBI-2880271\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"I6L9F6\", \"geneName\": \"NEFL\", \"intActId\": \"EBI-10178578\"}, \"numberOfExperiments\": 6, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P21359\", \"geneName\": \"NF1\", \"intActId\": \"EBI-1172917\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P01138\", \"geneName\": \"NGF\", \"intActId\": \"EBI-1028250\"}, \"numberOfExperiments\": 9, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P08138\", \"geneName\": \"NGFR\", \"intActId\": \"EBI-1387782\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q6IAD4\", \"geneName\": \"NOTCH1\", \"intActId\": \"EBI-25860267\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q99466\", \"geneName\": \"NOTCH4\", \"intActId\": \"EBI-7970822\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P43354\", \"geneName\": \"NR4A2\", \"intActId\": \"EBI-2681738\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q6PK61\", \"geneName\": \"NRG1\", \"intActId\": \"EBI-25938844\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q02818\", \"geneName\": \"NUCB1\", \"intActId\": \"EBI-2622179\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P49757-8\", \"geneName\": \"NUMB\", \"intActId\": \"EBI-25937715\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P04181\", \"geneName\": \"OAT\", \"intActId\": \"EBI-721662\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q96FW1\", \"geneName\": \"OTUB1\", \"intActId\": \"EBI-1058491\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P11940\", \"geneName\": \"PABPC1\", \"intActId\": \"EBI-81531\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O96013-2\", \"geneName\": \"PAK4\", \"intActId\": \"EBI-21659863\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q99497\", \"geneName\": \"PARK7\", \"intActId\": \"EBI-1164361\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q6ZW49\", \"geneName\": \"PAXIP1\", \"intActId\": \"EBI-743225\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P61457\", \"geneName\": \"PCBD1\", \"intActId\": \"EBI-740475\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P16234-2\", \"geneName\": \"PDGFRA\", \"intActId\": \"EBI-13380852\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P09619\", \"geneName\": \"PDGFRB\", \"intActId\": \"EBI-641237\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P30101\", \"geneName\": \"PDIA3\", \"intActId\": \"EBI-979862\"}, \"numberOfExperiments\": 6, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q15084\", \"geneName\": \"PDIA6\", \"intActId\": \"EBI-1043087\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q15118\", \"geneName\": \"PDK1\", \"intActId\": \"EBI-7016221\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q13113\", \"geneName\": \"PDZK1IP1\", \"intActId\": \"EBI-716063\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P18669\", \"geneName\": \"PGAM1\", \"intActId\": \"EBI-717905\"}, \"numberOfExperiments\": 4, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8WUB8-2\", \"geneName\": \"PHF10\", \"intActId\": \"EBI-10276329\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8N2W9\", \"geneName\": \"PIAS4\", \"intActId\": \"EBI-473160\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P42338\", \"geneName\": \"PIK3CB\", \"intActId\": \"EBI-2609540\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P48736\", \"geneName\": \"PIK3CG\", \"intActId\": \"EBI-1030384\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P27986-2\", \"geneName\": \"PIK3R1\", \"intActId\": \"EBI-9090282\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q13526\", \"geneName\": \"PIN1\", \"intActId\": \"EBI-714158\"}, \"numberOfExperiments\": 4, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9BXM7\", \"geneName\": \"PINK1\", \"intActId\": \"EBI-2846068\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q16512\", \"geneName\": \"PKN1\", \"intActId\": \"EBI-602382\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P00749\", \"geneName\": \"PLAU\", \"intActId\": \"EBI-3905042\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q13393\", \"geneName\": \"PLD1\", \"intActId\": \"EBI-2827556\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O14939\", \"geneName\": \"PLD2\", \"intActId\": \"EBI-1053996\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P53350\", \"geneName\": \"PLK1\", \"intActId\": \"EBI-476768\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O14494\", \"geneName\": \"PLPP1\", \"intActId\": \"EBI-2865290\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O15162\", \"geneName\": \"PLSCR1\", \"intActId\": \"EBI-740019\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8WVK1\", \"geneName\": \"PLSCR1\", \"intActId\": \"EBI-10238872\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"A0A6Q8PF08\", \"geneName\": \"PMP22\", \"intActId\": \"EBI-50433196\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P00491\", \"geneName\": \"PNP\", \"intActId\": \"EBI-712238\"}, \"numberOfExperiments\": 9, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P62937\", \"geneName\": \"PPIA\", \"intActId\": \"EBI-437708\"}, \"numberOfExperiments\": 4, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P62136\", \"geneName\": \"PPP1CA\", \"intActId\": \"EBI-357253\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P41236\", \"geneName\": \"PPP1R2\", \"intActId\": \"EBI-1056517\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P67775\", \"geneName\": \"PPP2CA\", \"intActId\": \"EBI-712311\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P63151\", \"geneName\": \"PPP2R2A\", \"intActId\": \"EBI-1048931\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q00005\", \"geneName\": \"PPP2R2B\", \"intActId\": \"EBI-1052159\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q15172\", \"geneName\": \"PPP2R5A\", \"intActId\": \"EBI-641666\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P48454\", \"geneName\": \"PPP3CC\", \"intActId\": \"EBI-2827192\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P17612\", \"geneName\": \"PRKACA\", \"intActId\": \"EBI-476586\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P22694\", \"geneName\": \"PRKACB\", \"intActId\": \"EBI-2679622\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P22694-8\", \"geneName\": \"PRKACB\", \"intActId\": \"EBI-25937151\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P22612\", \"geneName\": \"PRKACG\", \"intActId\": \"EBI-3907086\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9UGJ0-3\", \"geneName\": \"PRKAG2\", \"intActId\": \"EBI-25939641\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q05655\", \"geneName\": \"PRKCD\", \"intActId\": \"EBI-704279\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q02156\", \"geneName\": \"PRKCE\", \"intActId\": \"EBI-706254\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O60260-5\", \"geneName\": \"PRKN\", \"intActId\": \"EBI-21251460\"}, \"numberOfExperiments\": 5, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P04156\", \"geneName\": \"PRNP\", \"intActId\": \"EBI-977302\"}, \"numberOfExperiments\": 6, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P60891\", \"geneName\": \"PRPS1\", \"intActId\": \"EBI-749195\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P07602\", \"geneName\": \"PSAP\", \"intActId\": \"EBI-716699\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P49768\", \"geneName\": \"PSEN1\", \"intActId\": \"EBI-297277\"}, \"numberOfExperiments\": 6, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P49768-2\", \"geneName\": \"PSEN1\", \"intActId\": \"EBI-11047108\"}, \"numberOfExperiments\": 6, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P49810\", \"geneName\": \"PSEN2\", \"intActId\": \"EBI-2010251\"}, \"numberOfExperiments\": 4, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9NZ42\", \"geneName\": \"PSENEN\", \"intActId\": \"EBI-998468\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P28062-2\", \"geneName\": \"PSMB8\", \"intActId\": \"EBI-372312\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P17980\", \"geneName\": \"PSMC3\", \"intActId\": \"EBI-359720\"}, \"numberOfExperiments\": 6, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q14289\", \"geneName\": \"PTK2B\", \"intActId\": \"EBI-298640\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P20340-2\", \"geneName\": \"RAB6A\", \"intActId\": \"EBI-8840191\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P63000\", \"geneName\": \"RAC1\", \"intActId\": \"EBI-413628\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P04049\", \"geneName\": \"RAF1\", \"intActId\": \"EBI-365996\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q96S59\", \"geneName\": \"RANBP9\", \"intActId\": \"EBI-636085\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9Y272\", \"geneName\": \"RASD1\", \"intActId\": \"EBI-740818\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P61586\", \"geneName\": \"RHOA\", \"intActId\": \"EBI-446668\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9Y3C5\", \"geneName\": \"RNF11\", \"intActId\": \"EBI-396669\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q6ZNA4-2\", \"geneName\": \"RNF111\", \"intActId\": \"EBI-21535400\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9ULX5\", \"geneName\": \"RNF112\", \"intActId\": \"EBI-25829984\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O75116\", \"geneName\": \"ROCK2\", \"intActId\": \"EBI-366288\"}, \"numberOfExperiments\": 6, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P46779\", \"geneName\": \"RPL28\", \"intActId\": \"EBI-366357\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q15349\", \"geneName\": \"RPS6KA2\", \"intActId\": \"EBI-1384149\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P23443-4\", \"geneName\": \"RPS6KB1\", \"intActId\": \"EBI-25882353\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P04271\", \"geneName\": \"S100B\", \"intActId\": \"EBI-458391\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P21673\", \"geneName\": \"SAT1\", \"intActId\": \"EBI-711613\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q6AZY7-2\", \"geneName\": \"SCARA3\", \"intActId\": \"EBI-21598366\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8WTV0\", \"geneName\": \"SCARB1\", \"intActId\": \"EBI-78657\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P18827\", \"geneName\": \"SDC1\", \"intActId\": \"EBI-2855248\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q15019-3\", \"geneName\": \"SEPTIN2\", \"intActId\": \"EBI-11525407\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O43236\", \"geneName\": \"SEPTIN4\", \"intActId\": \"EBI-1047513\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q99719\", \"geneName\": \"SEPTIN5\", \"intActId\": \"EBI-373345\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q92599-3\", \"geneName\": \"SEPTIN8\", \"intActId\": \"EBI-25891137\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P01011\", \"geneName\": \"SERPINA3\", \"intActId\": \"EBI-296557\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P29353\", \"geneName\": \"SHC1\", \"intActId\": \"EBI-78835\"}, \"numberOfExperiments\": 5, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q92529\", \"geneName\": \"SHC3\", \"intActId\": \"EBI-79084\"}, \"numberOfExperiments\": 5, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8IUQ4-2\", \"geneName\": \"SIAH1\", \"intActId\": \"EBI-11522811\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9GZS3\", \"geneName\": \"SKIC8\", \"intActId\": \"EBI-358545\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q7Z2H8\", \"geneName\": \"SLC36A1\", \"intActId\": \"EBI-9978258\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9NP59\", \"geneName\": \"SLC40A1\", \"intActId\": \"EBI-725153\"}, \"numberOfExperiments\": 5, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P84022\", \"geneName\": \"SMAD3\", \"intActId\": \"EBI-347161\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q13485\", \"geneName\": \"SMAD4\", \"intActId\": \"EBI-347263\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P37840\", \"geneName\": \"SNCA\", \"intActId\": \"EBI-985879\"}, \"numberOfExperiments\": 6, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q16143\", \"geneName\": \"SNCB\", \"intActId\": \"EBI-727106\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q15036\", \"geneName\": \"SNX17\", \"intActId\": \"EBI-1752620\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O60749\", \"geneName\": \"SNX2\", \"intActId\": \"EBI-1046690\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8WV41\", \"geneName\": \"SNX33\", \"intActId\": \"EBI-2481535\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9UNH7\", \"geneName\": \"SNX6\", \"intActId\": \"EBI-949294\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q92673\", \"geneName\": \"SORL1\", \"intActId\": \"EBI-1171329\"}, \"numberOfExperiments\": 5, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q99932-2\", \"geneName\": \"SPAG8\", \"intActId\": \"EBI-11959123\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P11277\", \"geneName\": \"SPTB\", \"intActId\": \"EBI-514908\"}, \"numberOfExperiments\": 6, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q13501\", \"geneName\": \"SQSTM1\", \"intActId\": \"EBI-307104\"}, \"numberOfExperiments\": 6, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P61278\", \"geneName\": \"SST\", \"intActId\": \"EBI-20823968\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P32745\", \"geneName\": \"SSTR3\", \"intActId\": \"EBI-6266935\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P40763-2\", \"geneName\": \"STAT3\", \"intActId\": \"EBI-10692009\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8IWL8\", \"geneName\": \"STH\", \"intActId\": \"EBI-12843506\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O14662-5\", \"geneName\": \"STX16\", \"intActId\": \"EBI-9089968\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q13190-4\", \"geneName\": \"STX5\", \"intActId\": \"EBI-25938350\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O43752\", \"geneName\": \"STX6\", \"intActId\": \"EBI-2695795\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P61764\", \"geneName\": \"STXBP1\", \"intActId\": \"EBI-960169\"}, \"numberOfExperiments\": 7, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9Y5B9\", \"geneName\": \"SUPT16H\", \"intActId\": \"EBI-1046849\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P43405\", \"geneName\": \"SYK\", \"intActId\": \"EBI-78302\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P43405-2\", \"geneName\": \"SYK\", \"intActId\": \"EBI-25892332\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P08247\", \"geneName\": \"SYP\", \"intActId\": \"EBI-9071725\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q13148\", \"geneName\": \"TARDBP\", \"intActId\": \"EBI-372899\"}, \"numberOfExperiments\": 6, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P20226\", \"geneName\": \"TBP\", \"intActId\": \"EBI-355371\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q16650\", \"geneName\": \"TBR1\", \"intActId\": \"EBI-1047158\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O43680\", \"geneName\": \"TCF21\", \"intActId\": \"EBI-723267\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P01137\", \"geneName\": \"TGFB1\", \"intActId\": \"EBI-779636\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P61812\", \"geneName\": \"TGFB2\", \"intActId\": \"EBI-779581\"}, \"numberOfExperiments\": 7, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q15583\", \"geneName\": \"TGIF1\", \"intActId\": \"EBI-714215\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q15583-2\", \"geneName\": \"TGIF1\", \"intActId\": \"EBI-12691451\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P04216\", \"geneName\": \"THY1\", \"intActId\": \"EBI-9071715\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P04183\", \"geneName\": \"TK1\", \"intActId\": \"EBI-712550\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9BX74\", \"geneName\": \"TM2D1\", \"intActId\": \"EBI-25832057\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P49755\", \"geneName\": \"TMED10\", \"intActId\": \"EBI-998422\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9BTD3\", \"geneName\": \"TMEM121\", \"intActId\": \"EBI-12155101\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P62328\", \"geneName\": \"TMSB4X\", \"intActId\": \"EBI-712598\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P01375\", \"geneName\": \"TNF\", \"intActId\": \"EBI-359977\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O43508\", \"geneName\": \"TNFSF12\", \"intActId\": \"EBI-6932080\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O75888-3\", \"geneName\": \"TNFSF13\", \"intActId\": \"EBI-12856452\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q96GM8\", \"geneName\": \"TOE1\", \"intActId\": \"EBI-717460\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O14656\", \"geneName\": \"TOR1A\", \"intActId\": \"EBI-524257\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O14656-2\", \"geneName\": \"TOR1A\", \"intActId\": \"EBI-25847109\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q05BL1\", \"geneName\": \"TP53BP2\", \"intActId\": \"EBI-11952721\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q13625\", \"geneName\": \"TP53BP2\", \"intActId\": \"EBI-77642\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9C026\", \"geneName\": \"TRIM9\", \"intActId\": \"EBI-720828\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q15714-2\", \"geneName\": \"TSC22D1\", \"intActId\": \"EBI-12034704\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P02766\", \"geneName\": \"TTR\", \"intActId\": \"EBI-711909\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q71U36\", \"geneName\": \"TUBA1A\", \"intActId\": \"EBI-302552\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P68363\", \"geneName\": \"TUBA1B\", \"intActId\": \"EBI-487083\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P68366\", \"geneName\": \"TUBA4A\", \"intActId\": \"EBI-351772\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P07437\", \"geneName\": \"TUBB\", \"intActId\": \"EBI-350864\"}, \"numberOfExperiments\": 5, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8TBC4\", \"geneName\": \"UBA3\", \"intActId\": \"EBI-717567\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P0CG47\", \"geneName\": \"UBB\", \"intActId\": \"EBI-413034\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P62837\", \"geneName\": \"UBE2D2\", \"intActId\": \"EBI-347677\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9UMX0\", \"geneName\": \"UBQLN1\", \"intActId\": \"EBI-741480\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P09936\", \"geneName\": \"UCHL1\", \"intActId\": \"EBI-714860\"}, \"numberOfExperiments\": 5, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P13051-2\", \"geneName\": \"UNG\", \"intActId\": \"EBI-25834258\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O75604-3\", \"geneName\": \"USP2\", \"intActId\": \"EBI-10696113\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9BVJ6\", \"geneName\": \"UTP14A\", \"intActId\": \"EBI-473284\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9H270\", \"geneName\": \"VPS11\", \"intActId\": \"EBI-373380\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8N0S8\", \"geneName\": \"VPS29\", \"intActId\": \"EBI-25892084\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q96AX1\", \"geneName\": \"VPS33A\", \"intActId\": \"EBI-2527283\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q96QK1\", \"geneName\": \"VPS35\", \"intActId\": \"EBI-1054634\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O76024\", \"geneName\": \"WFS1\", \"intActId\": \"EBI-720609\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O00744\", \"geneName\": \"WNT10B\", \"intActId\": \"EBI-21797207\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P19544-6\", \"geneName\": \"WT1\", \"intActId\": \"EBI-11745701\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P31946\", \"geneName\": \"YWHAB\", \"intActId\": \"EBI-359815\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O60293\", \"geneName\": \"ZFC3H1\", \"intActId\": \"EBI-746701\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P17028\", \"geneName\": \"ZNF24\", \"intActId\": \"EBI-707773\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8N895\", \"geneName\": \"ZNF366\", \"intActId\": \"EBI-2813661\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8NHT4\", \"intActId\": \"EBI-25939025\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O35431\", \"geneName\": \"Apba2\", \"intActId\": \"EBI-2028211\"}, \"numberOfExperiments\": 5, \"organismDiffer\": true}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P15253\", \"geneName\": \"CALR\", \"intActId\": \"EBI-9005200\"}, \"numberOfExperiments\": 3, \"organismDiffer\": true}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8BGY9\", \"geneName\": \"Slc5a7\", \"intActId\": \"EBI-2010752\"}, \"numberOfExperiments\": 2, \"organismDiffer\": true}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067\", \"intActId\": \"EBI-77613\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q306T3\", \"intActId\": \"EBI-8294101\"}, \"numberOfExperiments\": 3, \"organismDiffer\": true}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9H7C9\", \"geneName\": \"AAMDC\", \"intActId\": \"EBI-10308705\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P63010-2\", \"geneName\": \"AP2B1\", \"intActId\": \"EBI-11529439\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q0P5N6\", \"geneName\": \"ARL16\", \"intActId\": \"EBI-10186132\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O15392\", \"geneName\": \"BIRC5\", \"intActId\": \"EBI-518823\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9UHY8\", \"geneName\": \"FEZ2\", \"intActId\": \"EBI-396453\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P06241-3\", \"geneName\": \"FYN\", \"intActId\": \"EBI-10691738\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q12891\", \"geneName\": \"HYAL2\", \"intActId\": \"EBI-2806068\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q6DN90-2\", \"geneName\": \"IQSEC1\", \"intActId\": \"EBI-21911304\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9BYQ4\", \"geneName\": \"KRTAP9-2\", \"intActId\": \"EBI-1044640\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O95447\", \"geneName\": \"LCA5L\", \"intActId\": \"EBI-8473670\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9BYZ2\", \"geneName\": \"LDHAL6B\", \"intActId\": \"EBI-1108377\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8TDB4\", \"geneName\": \"MGARP\", \"intActId\": \"EBI-4397720\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"A4FUJ8\", \"geneName\": \"MKL1\", \"intActId\": \"EBI-21250407\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P15941-11\", \"geneName\": \"MUC1\", \"intActId\": \"EBI-17263240\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q13113\", \"geneName\": \"PDZK1IP1\", \"intActId\": \"EBI-716063\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q6ZNA4-2\", \"geneName\": \"RNF111\", \"intActId\": \"EBI-21535400\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9ULX5\", \"geneName\": \"RNF112\", \"intActId\": \"EBI-25829984\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q2NKQ1-4\", \"geneName\": \"SGSM1\", \"intActId\": \"EBI-10182463\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8IUQ4-2\", \"geneName\": \"SIAH1\", \"intActId\": \"EBI-11522811\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9GZS3\", \"geneName\": \"SKIC8\", \"intActId\": \"EBI-358545\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q99932-2\", \"geneName\": \"SPAG8\", \"intActId\": \"EBI-11959123\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8IUW3\", \"geneName\": \"SPATA2L\", \"intActId\": \"EBI-2510414\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q13148\", \"geneName\": \"TARDBP\", \"intActId\": \"EBI-372899\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q16650\", \"geneName\": \"TBR1\", \"intActId\": \"EBI-1047158\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q5HYA8\", \"geneName\": \"TMEM67\", \"intActId\": \"EBI-11334880\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-2\", \"intActId\": \"EBI-17264467\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P09936\", \"geneName\": \"UCHL1\", \"intActId\": \"EBI-714860\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-4\", \"intActId\": \"EBI-302641\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O00213\", \"geneName\": \"APBB1\", \"intActId\": \"EBI-81694\"}, \"numberOfExperiments\": 5, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-4\", \"intActId\": \"EBI-302641\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P51693\", \"geneName\": \"APLP1\", \"intActId\": \"EBI-74648\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-4\", \"intActId\": \"EBI-302641\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q06481\", \"geneName\": \"APLP2\", \"intActId\": \"EBI-79306\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-4\", \"intActId\": \"EBI-302641\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P05067-4\", \"geneName\": \"APP\", \"intActId\": \"EBI-302641\"}, \"numberOfExperiments\": 8, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-4\", \"intActId\": \"EBI-302641\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q13867\", \"geneName\": \"BLMH\", \"intActId\": \"EBI-718504\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-4\", \"intActId\": \"EBI-302641\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9NZU0\", \"geneName\": \"FLRT3\", \"intActId\": \"EBI-1057092\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-4\", \"intActId\": \"EBI-302641\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P46089\", \"geneName\": \"GPR3\", \"intActId\": \"EBI-3909653\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-4\", \"intActId\": \"EBI-302641\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O43736\", \"geneName\": \"ITM2A\", \"intActId\": \"EBI-2431769\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-4\", \"intActId\": \"EBI-302641\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q68DU8\", \"geneName\": \"KCTD16\", \"intActId\": \"EBI-20768174\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-4\", \"intActId\": \"EBI-302641\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q96FE5\", \"geneName\": \"LINGO1\", \"intActId\": \"EBI-719955\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-4\", \"intActId\": \"EBI-302641\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P04629\", \"geneName\": \"NTRK1\", \"intActId\": \"EBI-1028226\"}, \"numberOfExperiments\": 7, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-4\", \"intActId\": \"EBI-302641\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q13526\", \"geneName\": \"PIN1\", \"intActId\": \"EBI-714158\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-4\", \"intActId\": \"EBI-302641\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P60201\", \"geneName\": \"PLP1\", \"intActId\": \"EBI-8653150\"}, \"numberOfExperiments\": 5, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-4\", \"intActId\": \"EBI-302641\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P04156\", \"geneName\": \"PRNP\", \"intActId\": \"EBI-977302\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-4\", \"intActId\": \"EBI-302641\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P49768\", \"geneName\": \"PSEN1\", \"intActId\": \"EBI-297277\"}, \"numberOfExperiments\": 4, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-4\", \"intActId\": \"EBI-302641\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q92673\", \"geneName\": \"SORL1\", \"intActId\": \"EBI-1171329\"}, \"numberOfExperiments\": 8, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-4\", \"intActId\": \"EBI-302641\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q99523\", \"geneName\": \"SORT1\", \"chainId\": \"PRO_0000033163\", \"intActId\": \"EBI-21467118\"}, \"numberOfExperiments\": 4, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-4\", \"intActId\": \"EBI-302641\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9HCB6\", \"geneName\": \"SPON1\", \"intActId\": \"EBI-2431846\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-4\", \"intActId\": \"EBI-302641\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O95793\", \"geneName\": \"STAU1\", \"intActId\": \"EBI-358174\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-4\", \"intActId\": \"EBI-302641\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8VEK0\", \"geneName\": \"Tmem30a\", \"intActId\": \"EBI-8381028\"}, \"numberOfExperiments\": 6, \"organismDiffer\": true}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-8\", \"intActId\": \"EBI-302661\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P17677\", \"geneName\": \"GAP43\", \"intActId\": \"EBI-1267511\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-8\", \"intActId\": \"EBI-302661\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9NSC5\", \"geneName\": \"HOMER3\", \"intActId\": \"EBI-748420\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P05067-8\", \"intActId\": \"EBI-302661\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9Y287\", \"geneName\": \"ITM2B\", \"intActId\": \"EBI-2866431\"}, \"numberOfExperiments\": 4, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000089\", \"intActId\": \"EBI-20829246\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O95631\", \"geneName\": \"NTN1\", \"intActId\": \"EBI-2678626\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000090\", \"intActId\": \"EBI-21194918\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9UIK5\", \"geneName\": \"TMEFF2\", \"intActId\": \"EBI-11423693\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000091\", \"intActId\": \"EBI-3894543\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q92673\", \"geneName\": \"SORL1\", \"intActId\": \"EBI-1171329\"}, \"numberOfExperiments\": 4, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000091\", \"intActId\": \"EBI-3894543\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8K3H7\", \"geneName\": \"CALR\", \"intActId\": \"EBI-9005068\"}, \"numberOfExperiments\": 2, \"organismDiffer\": true}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000091\", \"intActId\": \"EBI-3894543\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8VEK0\", \"geneName\": \"Tmem30a\", \"intActId\": \"EBI-8381028\"}, \"numberOfExperiments\": 3, \"organismDiffer\": true}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9BYF1\", \"geneName\": \"ACE2\", \"intActId\": \"EBI-7730807\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P05067\", \"geneName\": \"APP\", \"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"numberOfExperiments\": 77, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P48047\", \"geneName\": \"ATP5PO\", \"intActId\": \"EBI-355815\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P36544\", \"geneName\": \"CHRNA7\", \"intActId\": \"EBI-79333\"}, \"numberOfExperiments\": 7, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P10909-5\", \"geneName\": \"CLU\", \"intActId\": \"EBI-10961636\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P39060\", \"geneName\": \"COL18A1\", \"chainId\": \"PRO_0000005794\", \"intActId\": \"EBI-2566375\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O00230\", \"geneName\": \"CORT\", \"chainId\": \"PRO_0000033156\", \"intActId\": \"EBI-20824092\"}, \"numberOfExperiments\": 4, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q99714\", \"geneName\": \"HSD17B10\", \"intActId\": \"EBI-79964\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8N423\", \"geneName\": \"LILRB2\", \"intActId\": \"EBI-2816428\"}, \"numberOfExperiments\": 7, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P10636\", \"geneName\": \"MAPT\", \"intActId\": \"EBI-366182\"}, \"numberOfExperiments\": 5, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P08253\", \"geneName\": \"MMP2\", \"intActId\": \"EBI-1033518\"}, \"numberOfExperiments\": 4, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9NZV6\", \"geneName\": \"MSRB1\", \"intActId\": \"EBI-12330065\"}, \"numberOfExperiments\": 4, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P03897\", \"geneName\": \"MT-ND3\", \"intActId\": \"EBI-1246249\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8IVG9\", \"geneName\": \"MT-RNR2\", \"intActId\": \"EBI-8643752\"}, \"numberOfExperiments\": 4, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O95411\", \"geneName\": \"MYO18A\", \"intActId\": \"EBI-302378\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O95631\", \"geneName\": \"NTN1\", \"intActId\": \"EBI-2678626\"}, \"numberOfExperiments\": 6, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q15113\", \"geneName\": \"PCOLCE\", \"intActId\": \"EBI-8869614\"}, \"numberOfExperiments\": 4, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q08752\", \"geneName\": \"PPID\", \"intActId\": \"EBI-716596\"}, \"numberOfExperiments\": 4, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P30405\", \"geneName\": \"PPIF\", \"intActId\": \"EBI-5544229\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P04156\", \"geneName\": \"PRNP\", \"intActId\": \"EBI-977302\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P11686-1\", \"geneName\": \"SFTPC\", \"intActId\": \"EBI-16143688\"}, \"numberOfExperiments\": 5, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P61278\", \"geneName\": \"SST\", \"chainId\": \"PRO_0000033088\", \"intActId\": \"EBI-20824010\"}, \"numberOfExperiments\": 8, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P21980\", \"geneName\": \"TGM2\", \"intActId\": \"EBI-727668\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O60602\", \"geneName\": \"TLR5\", \"intActId\": \"EBI-3505951\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9NZC2\", \"geneName\": \"TREM2\", \"intActId\": \"EBI-14036387\"}, \"numberOfExperiments\": 4, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P02766\", \"geneName\": \"TTR\", \"intActId\": \"EBI-711909\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P15253\", \"geneName\": \"CALR\", \"intActId\": \"EBI-9005200\"}, \"numberOfExperiments\": 2, \"organismDiffer\": true}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q05941\", \"geneName\": \"Chrna7\", \"intActId\": \"EBI-79422\"}, \"numberOfExperiments\": 3, \"organismDiffer\": true}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P03452\", \"geneName\": \"HA\", \"intActId\": \"EBI-2548105\"}, \"numberOfExperiments\": 2, \"organismDiffer\": true}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P97484\", \"geneName\": \"Lilrb3\", \"intActId\": \"EBI-15728641\"}, \"numberOfExperiments\": 8, \"organismDiffer\": true}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"K9N5Q8\", \"geneName\": \"S\", \"intActId\": \"EBI-25474996\"}, \"numberOfExperiments\": 2, \"organismDiffer\": true}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P0DTC2\", \"geneName\": \"S\", \"chainId\": \"PRO_0000449647\", \"intActId\": \"EBI-25490323\"}, \"numberOfExperiments\": 3, \"organismDiffer\": true}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000092\", \"intActId\": \"EBI-821758\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q99NH8\", \"geneName\": \"Trem2\", \"intActId\": \"EBI-15982016\"}, \"numberOfExperiments\": 2, \"organismDiffer\": true}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000093\", \"intActId\": \"EBI-2431589\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P02649\", \"geneName\": \"APOE\", \"intActId\": \"EBI-1222467\"}, \"numberOfExperiments\": 4, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000093\", \"intActId\": \"EBI-2431589\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P05067\", \"geneName\": \"APP\", \"chainId\": \"PRO_0000000093\", \"intActId\": \"EBI-2431589\"}, \"numberOfExperiments\": 29, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000093\", \"intActId\": \"EBI-2431589\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P10909\", \"geneName\": \"CLU\", \"intActId\": \"EBI-1104674\"}, \"numberOfExperiments\": 4, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000093\", \"intActId\": \"EBI-2431589\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P49840\", \"geneName\": \"GSK3A\", \"intActId\": \"EBI-1044067\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000093\", \"intActId\": \"EBI-2431589\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P49841\", \"geneName\": \"GSK3B\", \"intActId\": \"EBI-373586\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000093\", \"intActId\": \"EBI-2431589\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P14735-1\", \"geneName\": \"IDE\", \"intActId\": \"EBI-15607031\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000093\", \"intActId\": \"EBI-2431589\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P08253\", \"geneName\": \"MMP2\", \"intActId\": \"EBI-1033518\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000093\", \"intActId\": \"EBI-2431589\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P08138\", \"geneName\": \"NGFR\", \"intActId\": \"EBI-1387782\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000093\", \"intActId\": \"EBI-2431589\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q5JRX3-1\", \"geneName\": \"PITRM1\", \"intActId\": \"EBI-16109799\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000093\", \"intActId\": \"EBI-2431589\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q08752\", \"geneName\": \"PPID\", \"intActId\": \"EBI-716596\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000093\", \"intActId\": \"EBI-2431589\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q92673\", \"geneName\": \"SORL1\", \"intActId\": \"EBI-1171329\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000093\", \"intActId\": \"EBI-2431589\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"O60602\", \"geneName\": \"TLR5\", \"intActId\": \"EBI-3505951\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000093\", \"intActId\": \"EBI-2431589\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P31696\", \"geneName\": \"AGRN\", \"intActId\": \"EBI-457650\"}, \"numberOfExperiments\": 3, \"organismDiffer\": true}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000093\", \"intActId\": \"EBI-2431589\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P15253\", \"geneName\": \"CALR\", \"intActId\": \"EBI-9005200\"}, \"numberOfExperiments\": 2, \"organismDiffer\": true}\n",
      "{\"interactantOne\": {\"chainId\": \"PRO_0000000093\", \"intActId\": \"EBI-2431589\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P07174\", \"geneName\": \"Ngfr\", \"intActId\": \"EBI-1038810\"}, \"numberOfExperiments\": 2, \"organismDiffer\": true}\n",
      "PATHWAYS:\n",
      "\n",
      "<END_CONTEXT>\n",
      "\n",
      "<BEGIN_CONTEXT>ABSTRACTS:\n",
      "Functional and morphological analyses indicated that the epithelial Ca2+ channel (ECaC), which was recently cloned from rabbit kidney, exhibits the defining properties for being the gatekeeper in transcellular Ca2+ (re)absorption. Its human homologue provides, therefore, a molecular basis for achieving a better understanding of Ca2+ mal(re)absorption. By applying the RACE technique, the full-length cDNA of human ECaC (HGMW-approved symbol ECAC1) was obtained. It consisted of 2,772 bp with an open reading frame of 2,187 bp encoding a protein of 729 amino acids with a predicted molecular mass of 83 kDa. Phylogenetic analysis indicated that this highly selective Ca2+ channel exhibits a low level of homology (<30%) to other Ca2+ channels, suggesting that it belongs to a new family. hECaC was highly expressed in kidney, small intestine, and pancreas, and less intense expression was detected in testis, prostate, placenta, brain, colon, and rectum. These ECaC-positive tissues also expressed the 1,25-dihydroxyvitamin D3-sensitive calcium-binding proteins, calbindin-D9K and/or calbindin-D28K. The human ECaC gene mapped to chromosome 7q31.1-q31.2. Taken together, the conspicuous colocalization of hECaC and calbindins in organs that are not prime regulators of plasma Ca2+ levels could illustrate new pathways in cellular Ca2+ homeostasis.\n",
      "We report here the genomic structures of the genes encoding human calcium transport proteins CaT1 and CaT2, which belong to a recently identified class of highly selective calcium entry channels. The mRNA for CaT1 was expressed more abundantly than that for CaT2 in three major tissues involved in transcellular calcium transport, namely intestine, kidney, and placenta, as determined by quantitative PCR. The genes encoding CaT1 and CaT2, ECAC2 and ECAC1, respectively, are completely conserved in terms of exon size in the coding regions. They also share similar intron-exon structures with the genes encoding the closely related, nonselective cation channels VR1, VRL-1, OTRPC4 (also known as VR-OAC, Trp12, and VRL-2), and a hypothetical protein, VRL-3. We conclude that ECAC2 and ECAC1, which encode calcium selective channels, share a common ancestral gene with the genes encoding the related nonselective cation channels.\n",
      "Human chromosome 7 has historically received prominent attention in the human genetics community, primarily related to the search for the cystic fibrosis gene and the frequent cytogenetic changes associated with various forms of cancer. Here we present more than 153 million base pairs representing 99.4% of the euchromatic sequence of chromosome 7, the first metacentric chromosome completed so far. The sequence has excellent concordance with previously established physical and genetic maps, and it exhibits an unusual amount of segmentally duplicated sequence (8.2%), with marked differences between the two arms. Our initial analyses have identified 1,150 protein-coding genes, 605 of which have been confirmed by complementary DNA sequences, and an additional 941 pseudogenes. Of genes confirmed by transcript sequences, some are polymorphic for mutations that disrupt the reading frame.\n",
      "DNA sequence and annotation of the entire human chromosome 7, encompassing nearly 158 million nucleotides of DNA and 1917 gene structures, are presented. To generate a higher order description, additional structural features such as imprinted genes, fragile sites, and segmental duplications were integrated at the level of the DNA sequence with medical genetic data, including 440 chromosome rearrangement breakpoints associated with disease. This approach enabled the discovery of candidate genes for developmental diseases including autism.\n",
      "The National Institutes of Health's Mammalian Gene Collection (MGC) project was designed to generate and sequence a publicly accessible cDNA resource containing a complete open reading frame (ORF) for every human and mouse gene. The project initially used a random strategy to select clones from a large number of cDNA libraries from diverse tissues. Candidate clones were chosen based on 5'-EST sequences, and then fully sequenced to high accuracy and analyzed by algorithms developed for this project. Currently, more than 11,000 human and 10,000 mouse genes are represented in MGC by at least one clone with a full ORF. The random selection approach is now reaching a saturation point, and a transition to protocols targeted at the missing transcripts is now required to complete the mouse and human collections. Comparison of the sequence of the MGC clones to reference genome sequences reveals that most cDNA clones are of very high sequence quality, although it is likely that some cDNAs may carry missense variants as a consequence of experimental artifact, such as PCR, cloning, or reverse transcriptase errors. Recently, a rat cDNA component was added to the project, and ongoing frog (Xenopus) and zebrafish (Danio) cDNA projects were expanded to take advantage of the high-throughput MGC pipeline.\n",
      "WNK3, a member of the With No Lysine (K) family of protein serine/threonine kinases, was shown to regulate members of the SLC12A family of cation-chloride cotransporters and the renal outer medullary K+ channel ROMK and Cl(-) channel SLC26A9. To evaluate the effect of WNK3 on TRPV5, a renal epithelial Ca2+ channel that serves as a gatekeeper for active Ca2+ reabsorption, WNK3 and TRPV5 were coexpressed in Xenopus laevis oocytes and the function and expression of TRPV5 were subsequently examined. An 82.7 +/- 7.1% increase in TRPV5-mediated Ca2+ uptake was observed when WNK3 was coexpressed. A similar increase in TRPV5-mediated Na+ current was observed with the voltage-clamp technique. WNK3 also enhanced Ca2+ influx and Na+ current mediated by TRPV6, which is the closest homolog of TRPV5 that mediates active intestinal Ca2+ absorption. The kinase domain of WNK3 alone was sufficient to increase TRPV5-mediated Ca2+ transport, and the positive regulatory effect was abolished by the kinase-inactive D294A mutation in WNK3, indicating a kinase-dependent mechanism. The complexly glycosylated TRPV5 that appears at the plasma membrane was increased by WNK3. The exocytosis of TRPV5 was increased by WNK3, and the effect of WNK3 on TRPV5 was abolished by the microtubule inhibitor colchicine. The increased plasma membrane expression of TRPV5 was likely due to the enhanced delivery of mature TRPV5 to the plasma membrane from its intracellular pool via the secretory pathway. These results indicate that WNK3 is a positive regulator of the transcellular Ca2+ transport pathway.\n",
      "The eukaryotic cell division cycle is characterized by a sequence of orderly and highly regulated events resulting in the duplication and separation of all cellular material into two newly formed daughter cells. Protein phosphorylation by cyclin-dependent kinases (CDKs) drives this cycle. To gain further insight into how phosphorylation regulates the cell cycle, we sought to identify proteins whose phosphorylation is cell cycle regulated. Using stable isotope labeling along with a two-step strategy for phosphopeptide enrichment and high mass accuracy mass spectrometry, we examined protein phosphorylation in a human cell line arrested in the G(1) and mitotic phases of the cell cycle. We report the identification of >14,000 different phosphorylation events, more than half of which, to our knowledge, have not been described in the literature, along with relative quantitative data for the majority of these sites. We observed >1,000 proteins with increased phosphorylation in mitosis including many known cell cycle regulators. The majority of sites on regulated phosphopeptides lie in [S/T]P motifs, the minimum required sequence for CDKs, suggesting that many of the proteins may be CDK substrates. Analysis of non-proline site-containing phosphopeptides identified two unique motifs that suggest there are at least two undiscovered mitotic kinases.\n",
      "INTERACTIONS:\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"Q9NQA5\", \"intActId\": \"EBI-751281\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P05937\", \"geneName\": \"CALB1\", \"intActId\": \"EBI-4286943\"}, \"numberOfExperiments\": 4, \"organismDiffer\": false}\n",
      "PATHWAYS:\n",
      "\n",
      "<END_CONTEXT>\n",
      "\n",
      "<BEGIN_CONTEXT>ABSTRACTS:\n",
      "Transforming growth factor-beta (TGF-beta) is capable of affecting the proliferation of many cell types. To identify novel genes whose protein products may mediate cellular responses to this factor, a cDNA library was made from mRNA isolated from a human lung adenocarcinoma cell line (A549) that had been treated for 3 days with TGF-beta. The library was screened by differential hybridization and a cDNA clone, beta ig-h3, was isolated. This gene was induced up to 20-fold in A549 cells after 2 days of treatment with TGF-beta 1. It was also induced in several other cell lines, including PC-3 and H2981. DNA sequence analysis of beta ig-h3 indicated that it encoded a novel protein, beta IG-H3, of 683 amino acids, which contained an amino-terminal secretory sequence and a carboxy-terminal Arg-Gly-Asp (RGD) sequence that can serve as a ligand recognition site for several integrins. beta IG-H3 also contained short amino acid regions homologous to similar regions in Drosophila fasciclin-I and four homologous internal domains, which can be folded into a potential bivalent structure and could act as a bridge between cells expressing the appropriate ligand. beta ig-h3 RNA was detected in several cell lines and tissues. COS cells transfected with plasmids encoding beta IG-H3 secreted a major 68-kD protein that was detected by immunoblotting using antipeptide antibodies. Since beta ig-h3 is induced in several cell lines whose proliferation is affected by TGF-beta 1, it may be involved in mediating some of the signals of this multifunctional growth modulator.\n",
      "Granular dystrophy Groenouw type I (CDGG1), Reis-Bcklers (CDRB), lattice type I (CDL1) and Avellino (ACD) are four 5q31-linked human autosomal dominant corneal dystrophies. Clinically, they show progressive opacification of the cornea leading to severe visual handicap. The nature of the deposits remains unknown in spite of amyloid aetiology ascribed to the last two. We generated a YAC contig of the linked region and, following cDNA selection, recovered the beta ig-h3 gene. In six affected families we identified missense mutations. All detected mutations occurred at the CpG dinucleotide of two arginine codons: R555W in one CDGG1, R555Q in one CDRB, R124C in two CDL1 and R124H in two ACD families. This suggests, as the last two diseases are characterized by amyloid deposits, that R124 mutated kerato-epithelin (the product of beta ig-h3) forms amyloidogenic intermediates that precipitate in the cornea. Our data establish a common molecular origin for the 5q31-linked corneal dystrophies.\n",
      "Chromosome 5 is one of the largest human chromosomes and contains numerous intrachromosomal duplications, yet it has one of the lowest gene densities. This is partially explained by numerous gene-poor regions that display a remarkable degree of noncoding conservation with non-mammalian vertebrates, suggesting that they are functionally constrained. In total, we compiled 177.7 million base pairs of highly accurate finished sequence containing 923 manually curated protein-coding genes including the protocadherin and interleukin gene families. We also completely sequenced versions of the large chromosome-5-specific internal duplications. These duplications are very recent evolutionary events and probably have a mechanistic role in human physiological variation, as deletions in these regions are the cause of debilitating disorders including spinal muscular atrophy.\n",
      "The National Institutes of Health's Mammalian Gene Collection (MGC) project was designed to generate and sequence a publicly accessible cDNA resource containing a complete open reading frame (ORF) for every human and mouse gene. The project initially used a random strategy to select clones from a large number of cDNA libraries from diverse tissues. Candidate clones were chosen based on 5'-EST sequences, and then fully sequenced to high accuracy and analyzed by algorithms developed for this project. Currently, more than 11,000 human and 10,000 mouse genes are represented in MGC by at least one clone with a full ORF. The random selection approach is now reaching a saturation point, and a transition to protocols targeted at the missing transcripts is now required to complete the mouse and human collections. Comparison of the sequence of the MGC clones to reference genome sequences reveals that most cDNA clones are of very high sequence quality, although it is likely that some cDNAs may carry missense variants as a consequence of experimental artifact, such as PCR, cloning, or reverse transcriptase errors. Recently, a rat cDNA component was added to the project, and ongoing frog (Xenopus) and zebrafish (Danio) cDNA projects were expanded to take advantage of the high-throughput MGC pipeline.\n",
      "beta ig-h3 is a novel gene first discovered by differential screening of a cDNA library made from A549 human lung adenocarcinoma cells treated with transforming growth factor-beta 1 (TGF-beta 1). It encodes a 683-amino-acid protein containing a secretory signal sequence and four homologous internal domains. Here we show that treatment of several types of cells, including human melanoma cells, human mammary epithelial cells, human keratinocytes, and human fibroblasts, with TGF-beta resulted in a significant increase in beta ig-h3 RNA. A portion of the beta ig-h3 coding sequence was expressed in bacteria, and antisera against the bacterially produced protein was raised in rabbits. This antisera was used to demonstrate that several cell lines secreted a 68-kD beta IG-H3 protein after treatment with TGF-beta. Transfection of beta IG-H3 expression plasmids into Chinese hamster ovary (CHO) cells led to a marked decrease in the ability of these cells to form tumors in nude mice. The beta IG-H3 protein was purified from media conditioned by recombinant CHO cells, characterized by immunoblotting and protein sequencing and shown to function in an anti-adhesion assay in that it inhibited the attachment of A549, HeLa, and WI-38 cells to plastic in serum-free media. Sequencing of cDNA clones encoding murine beta ig-H3 indicated 90.6% conservation at the amino acid level between the murine and human proteins. Finally, the beta ig-h3 gene was localized to human chromosome 5q31, a region frequently deleted in preleukemic myelodysplasia and leukemia. The corresponding mouse beta ig-h3 gene was mapped to mouse chromosome 13 region B to C1, which confirms a region of conservation on human chromosome 5 and mouse chromosome 13. We suggest that this protein be named p68 beta ig-h3.\n",
      "Transforming growth factor -induced protein (TGFBIp) is an extracellular matrix protein composed of an NH<sub>2</sub>-terminal cysteine-rich domain (CRD) annotated as an emilin (EMI) domain and four fasciclin-1 (FAS1-1-FAS1-4) domains. Mutations in the gene cause corneal dystrophies, a group of debilitating protein misfolding diseases that lead to severe visual impairment. Previous studies have shown that TGFBIp in the cornea is cross-linked to type XII collagen through a reducible bond. TGFBIp contains 11 cysteine residues and is thus able to form five intramolecule disulfide bonds, leaving a single cysteine residue available for the collagen cross-link. The structures of TGFBIp and its homologues are unknown. We here present the disulfide bridge pattern of TGFBIp, which was determined by generating specific peptides. These were separated by ion exchange followed by reversed-phase high-performance liquid chromatography and analyzed by mass spectrometry and Edman degradation. The NH<sub>2</sub>-terminal CRD contains six cysteine residues, and one of these (Cys65) was identified as the candidate for the reducible cross-link between TGFBIp and type XII collagen. In addition, the CRD contained two intradomain disulfide bridges (Cys49-Cys85 and Cys84-Cys97) and one interdomain disulfide bridge to FAS1-2 (Cys74-Cys339). Significantly, this arrangement violates the predicted disulfide bridge pattern of an EMI domain. The cysteine residues in FAS1-3 (Cys473 and Cys478) were shown to form an intradomain disulfide bridge. Finally, an interdomain disulfide bridge between FAS1-1 and FAS1-2 (Cys214-Cys317) was identified. The interdomain disulfide bonds indicate that the NH<sub>2</sub> terminus of TGFBIp (CRD, FAS1-1, and FAS1-2) adopts a compact globular fold, leaving FAS1-3 and FAS1-4 exposed.\n",
      "The non-pigmented ciliary epithelium is largely responsible for the formation of aqueous humor in the mammalian eye. To provide a basis for studies at the molecular level, a directional expression cDNA library was constructed in Uni-ZAP XR vector from poly A+ RNA of the human non-pigmented ciliary epithelial derived ODM-2 cell line. Fifty-three cDNA clones were isolated from the library and characterized by partial sequence analysis. Approximately 49% of the clones exhibited homology with known genes in the GenBank/EMBL databases. The putative identification of these clones may reflect the transcriptional activity of the ODM-2 cells in culture. One of the identified clones, ODM-42-I, was found to be specific and highly expressed in the corneal epithelium. This clone had an exact match with a recently discovered human gene, beta ig-h3 (Skonier et al., 1992, DNA Cell Biol., 11:511-522), which codes a surface recognition protein, inducible by transforming growth factor beta (TGF-beta), and containing a putative binding site (RDG) for integrins. The ODM-42-I cDNA clone displays a distinctive pattern of expression found in the human eye, expressed almost exclusively in the cornea. Further studies, using sera from a synthetic peptide to the carboxy-terminal region of ODM-42-I, reveal that the protein is heterogeneous in charge and is preferentially expressed on the extracellular surface of corneal epithelial cells, and might share immunologic properties with integrins beta 1.\n",
      "Recent advances in molecular genetics have increased our understanding of the role of genes. Four autosomal dominant corneal dystrophies (CDs); granular CD (GCD), Avellino CD (ACD), lattice CD (LCD), and Reis-Bcklers CD (RBCD) were mapped to the long arm of chromosome 5 (5q31). These four diseases were shown, in a Caucasian series, to result from different missense mutations in the TGFBI (BIGH3, keratoepithelin) gene. The same mutations were also detected in Japanese patients, from a different ethnic background. Gelatinous drop-like corneal dystrophy (GDLD), on the other hand, which was found in Japanese patients in 1914, is a rare autosomal recessive disorder characterized by corneal amyloidosis. Parents of the patients had a markedly higher frequency of consanguineous marriages than the general population. The gene responsible for GDLD, the membrane component, chromosome 1, surface marker 1 (M1S1) gene was mapped to the short arm of chromosome 1(1p). Four deleterious mutations in this gene were detected in Japanese patients. We review here additional studies on mutations of the TGFBI and M1S1 genes found in Japanese patients. In the TGFBI gene, nine different mutations were detected in Japanese patients with GCD, ACD, LCD, or RBCD. The codons R124 and R555 of the TGFBI gene were hotspots in Japanese patients, of whom many were ACD patients with the R124H mutation. New mutations responsible for LCD were detected in the TGFBI gene of patients with LCD, in addition to the P501T mutation in LCD type IIIA found earlier. These studies showed a clear genotype/phenotype correlation associated with the TGFBI gene. In the M1S1 gene, the Q118X mutation was the most common alteration, and a founder mutation in Japanese GDLD patients, as previously reported. Ninety-two percent of the mutated alleles were the Q118X.\n",
      "The modification of glutamic acid residues to gamma-carboxyglutamic acid (Gla) is a post-translational modification catalyzed by the vitamin K-dependent enzyme gamma-glutamylcarboxylase. Despite ubiquitous expression of the gamma-carboxylation machinery in mammalian tissues, only 12 Gla-containing proteins have so far been identified in humans. Because bone tissue is the second most abundant source of Gla-containing proteins after the liver, we sought to identify Gla proteins secreted by bone marrow-derived mesenchymal stromal cells (MSCs). We used a proteomics approach to screen the secretome of MSCs with a combination of two-dimensional gel electrophoresis and tandem mass spectrometry. The most abundant Gla-containing protein secreted by MSCs was identified as periostin, a previously unrecognized gamma-carboxylated protein. In silico amino acid sequence analysis of periostin demonstrated the presence of four consensus gamma-carboxylase recognition sites embedded within fasciclin-like protein domains. The carboxylation of periostin was confirmed by immunoprecipitation and purification of the recombinant protein. Carboxylation of periostin could be inhibited by warfarin in MSCs, demonstrating its dependence on the presence of vitamin K. We were able to demonstrate localization of carboxylated periostin to bone nodules formed by MSCs in vitro, suggesting a role in extracellular matrix mineralization. Our data also show that another fasciclin I-like protein, betaig-h3, contains Gla. In conclusion, periostin is a member of a novel vitamin K-dependent gamma-carboxylated protein family characterized by the presence of fasciclin domains. Furthermore, carboxylated periostin is produced by bone-derived cells of mesenchymal lineage and is abundantly found in mineralized bone nodules in vitro.\n",
      "UNLABELLED: Protein phosphorylation is one of the most common post-translational modifications. It plays key roles in regulating diverse biological processes of liver tissues. To better understand the role of protein phosphorylation in liver functions, it is essential to perform in-depth phosphoproteome analysis of human liver. Here, an enzyme assisted reversed-phase-reversed-phase liquid chromatography (RP-RPLC) approach with both RPLC separations operated with optimized acidic mobile phase was developed. High orthogonal separation was achieved by trypsin digestion of the Glu-C generated peptides in the fractions collected from the first RPLC separation. The phosphoproteome coverage was further improved by using two types of instruments, i.e. TripleTOF 5600 and LTQ Orbitrap Velos. A total of 22,446 phosphorylation sites, corresponding to 6526 nonredundant phosphoproteins were finally identified from normal human liver tissues. Of these sites, 15,229 sites were confidently localized with Ascore13. This dataset was the largest phosphoproteome dataset of human liver. It can be a public resource for the liver research community and holds promise for further biology studies.\n",
      "BIOLOGICAL SIGNIFICANCE: The enzyme assisted approach enabled the two RPLC separations operated both with optimized acidic mobile phases. The identifications from TripleTOF 5600 and Orbitrap Velos are highly complementary. The largest phosphoproteome dataset of human liver was generated.\n",
      "INTERACTIONS:\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"Q15582\", \"intActId\": \"EBI-10236573\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8IZU0\", \"geneName\": \"FAM9B\", \"intActId\": \"EBI-10175124\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"Q15582\", \"intActId\": \"EBI-10236573\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P05107\", \"geneName\": \"ITGB2\", \"intActId\": \"EBI-300173\"}, \"numberOfExperiments\": 2, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"Q15582\", \"intActId\": \"EBI-10236573\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q15063\", \"geneName\": \"POSTN\", \"intActId\": \"EBI-7067070\"}, \"numberOfExperiments\": 7, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"Q15582\", \"intActId\": \"EBI-10236573\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P25815\", \"geneName\": \"S100P\", \"intActId\": \"EBI-743700\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"Q15582\", \"intActId\": \"EBI-10236573\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q15582\", \"geneName\": \"TGFBI\", \"intActId\": \"EBI-10236573\"}, \"numberOfExperiments\": 5, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"Q15582\", \"intActId\": \"EBI-10236573\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"A0A0G2K692\", \"geneName\": \"Postn\", \"intActId\": \"EBI-25473213\"}, \"numberOfExperiments\": 5, \"organismDiffer\": true}\n",
      "PATHWAYS:\n",
      "\n",
      "<END_CONTEXT>\n",
      "\n",
      "<BEGIN_CONTEXT>ABSTRACTS:\n",
      "Ribosomal protein L37 mRNA is overexpressed in colon cancer. The nucleotide sequences of human L37 from several tumor and normal, colon and liver cDNA sources were determined to be identical. L37 mRNA was approximately 375 nucleotides long encoding 97 amino acids with M(r) = 11,070, pI = 12.6, multiple potential serine/threonine phosphorylation sites and a zinc-finger domain. The human sequence is compared to other species.\n",
      "A unique human cDNA (hG1.16) that encodes a mRNA of 450 nucleotides was isolated from a subtractive library derived from HeLa cells. The relative expression level of hG1.16 during different cell-cycle phases was determined by Northern-blot analysis of cells synchronized by double-thymidine block and serum deprivation/refeeding. hG1.16 was constitutively expressed during all phases of the cell cycle, including the quiescent phase when even most constitutively expressed genes experience some suppression of expression. The expression level of hG1.16 did not change during terminal differentiation of myoblasts to myotubes, during which cells become permanently post-mitotic. Examination of other tissues revealed that the relative expression level of hG1.16 was constitutive in all embryonic mouse tissues examined, including brain, eye, heart, kidney, liver, lung and skeletal muscle. This was unusual in that expression was not down-modulated during differentiation and did not vary appreciably between tissue types. Analysis by inter-species Northern-blot analysis revealed that hG1.16 was highly conserved among all vertebrates studied (from fish to humans but not in insects). DNA sequence analysis of hG1.16 revealed a high level of similarity to rat ribosomal protein L37, identifying hG1.16 as a new member of this multigene family. The deduced amino acid sequence of hG1.16 was identical to rat ribosomal protein L37 that contained 97 amino acids, many of which are highly positively charged (15 arginine and 14 lysine residues with a predicted M(r) of 11,065). hG1.16 protein has a single C2-C2 zinc-finger-like motif which is also present in rat ribosomal protein L37. Using primers designed from the sequence of hG1.16, unique bovine and rat cDNAs were also isolated by 5'-rapid-amplification of cDNA ends. DNA sequences of bovine and rat G1.16, clones were 92.8% and 92.2% similar to human G1.16 while the deduced amino acid sequences derived from bovine and rat cDNAs each differed by a single amino acid from the sequence of hG1.16 and the published rat L37 sequence. Southern-blot analysis revealed that hG1.16 exists in multiple copies in human, rat and mouse genomes. These G1.16 clones encode unique human, rat and bovine members of the ribosomal protein L37 gene family, which are constitutively expressed even during transitions from quiescence to active cell proliferation or terminal differentiation, in all tissues and all vertebrates investigated.\n",
      "The ribosome, as a catalyst for protein synthesis, is universal and essential for all organisms. Here we describe the structure of the genes encoding human ribosomal proteins (RPs) and compare this class of genes among several eukaryotes. Using genomic and full-length cDNA sequences, we characterized 73 RP genes and found that (1) transcription starts at a C residue within a characteristic oligopyrimidine tract; (2) the promoter region is GC rich, but often has a TATA box or similar sequence element; (3) the genes are small (4.4 kb), but have as many as 5.6 exons on average; (4) the initiator ATG is in the first or second exon and is within plus minus 5 bp of the first intron boundaries in about half of cases; and (5) 5'- and 3'-UTRs are significantly smaller (42 bp and 56 bp, respectively) than the genome average. Comparison of RP genes from humans, Drosophila melanogaster, Caenorhabditis elegans, and Saccharomyces cerevisiae revealed the coding sequences to be highly conserved (63% homology on average), although gene size and the number of exons vary. The positions of the introns are also conserved among these species as follows: 44% of human introns are present at the same position in either D. melanogaster or C. elegans, suggesting RP genes are highly suitable for studying the evolution of introns.\n",
      "As a base for human transcriptome and functional genomics, we created the \"full-length long Japan\" (FLJ) collection of sequenced human cDNAs. We determined the entire sequence of 21,243 selected clones and found that 14,490 cDNAs (10,897 clusters) were unique to the FLJ collection. About half of them (5,416) seemed to be protein-coding. Of those, 1,999 clusters had not been predicted by computational methods. The distribution of GC content of nonpredicted cDNAs had a peak at approximately 58% compared with a peak at approximately 42%for predicted cDNAs. Thus, there seems to be a slight bias against GC-rich transcripts in current gene prediction procedures. The rest of the cDNAs unique to the FLJ collection (5,481) contained no obvious open reading frames (ORFs) and thus are candidate noncoding RNAs. About one-fourth of them (1,378) showed a clear pattern of splicing. The distribution of GC content of noncoding cDNAs was narrow and had a peak at approximately 42%, relatively low compared with that of protein-coding cDNAs.\n",
      "The National Institutes of Health's Mammalian Gene Collection (MGC) project was designed to generate and sequence a publicly accessible cDNA resource containing a complete open reading frame (ORF) for every human and mouse gene. The project initially used a random strategy to select clones from a large number of cDNA libraries from diverse tissues. Candidate clones were chosen based on 5'-EST sequences, and then fully sequenced to high accuracy and analyzed by algorithms developed for this project. Currently, more than 11,000 human and 10,000 mouse genes are represented in MGC by at least one clone with a full ORF. The random selection approach is now reaching a saturation point, and a transition to protocols targeted at the missing transcripts is now required to complete the mouse and human collections. Comparison of the sequence of the MGC clones to reference genome sequences reveals that most cDNA clones are of very high sequence quality, although it is likely that some cDNAs may carry missense variants as a consequence of experimental artifact, such as PCR, cloning, or reverse transcriptase errors. Recently, a rat cDNA component was added to the project, and ongoing frog (Xenopus) and zebrafish (Danio) cDNA projects were expanded to take advantage of the high-throughput MGC pipeline.\n",
      "We mapped 75 genes that collectively encode >90% of the proteins found in human ribosomes. Because localization of ribosomal protein genes (rp genes) is complicated by the existence of processed pseudogenes, multiple strategies were devised to identify PCR-detectable sequence-tagged sites (STSs) at introns. In some cases we exploited specific, pre-existing information about the intron/exon structure of a given human rp gene or its homolog in another vertebrate. When such information was unavailable, selection of PCR primer pairs was guided by general insights gleaned from analysis of all mammalian rp genes whose intron/exon structures have been published. For many genes, PCR amplification of introns was facilitated by use of YAC pool DNAs rather than total human genomic DNA as templates. We then assigned the rp gene STSs to individual human chromosomes by typing human-rodent hybrid cell lines. The genes were placed more precisely on the physical map of the human genome by typing of radiation hybrids or screening YAC libraries. Fifty-one previously unmapped rp genes were localized, and 24 previously reported rp gene localizations were confirmed, refined, or corrected. Though functionally related and coordinately expressed, the 75 mapped genes are widely dispersed: Both sex chromosomes and at least 20 of the 22 autosomes carry one or more rp genes. Chromosome 19, known to have a high gene density, contains an unusually large number of rp genes (12). This map provides a foundation for the study of the possible roles of ribosomal protein deficiencies in chromosomal and Mendelian disorders.\n",
      "Protein kinases are pivotal regulators of cell signaling that modulate each other's functions and activities through site-specific phosphorylation events. These key regulatory modifications have not been studied comprehensively, because low cellular abundance of kinases has resulted in their underrepresentation in previous phosphoproteome studies. Here, we combine kinase-selective affinity purification with quantitative mass spectrometry to analyze the cell-cycle regulation of protein kinases. This proteomics approach enabled us to quantify 219 protein kinases from S and M phase-arrested human cancer cells. We identified more than 1000 phosphorylation sites on protein kinases. Intriguingly, half of all kinase phosphopeptides were upregulated in mitosis. Our data reveal numerous unknown M phase-induced phosphorylation sites on kinases with established mitotic functions. We also find potential phosphorylation networks involving many protein kinases not previously implicated in mitotic progression. These results provide a vastly extended knowledge base for functional studies on kinases and their regulation through site-specific phosphorylation.\n",
      "Protein phosphorylation events during T cell receptor (TCR) signaling control the formation of complexes among proteins proximal to the TCR, the activation of kinase cascades, and the activation of transcription factors; however, the mode and extent of the influence of phosphorylation in coordinating the diverse phenomena associated with T cell activation are unclear. Therefore, we used the human Jurkat T cell leukemia cell line as a model system and performed large-scale quantitative phosphoproteomic analyses of TCR signaling. We identified 10,665 unique phosphorylation sites, of which 696 showed TCR-responsive changes. In addition, we analyzed broad trends in phosphorylation data sets to uncover underlying mechanisms associated with T cell activation. We found that, upon stimulation of the TCR, phosphorylation events extensively targeted protein modules involved in all of the salient phenomena associated with T cell activation: patterning of surface proteins, endocytosis of the TCR, formation of the F-actin cup, inside-out activation of integrins, polarization of microtubules, production of cytokines, and alternative splicing of messenger RNA. Further, case-by-case analysis of TCR-responsive phosphorylation sites on proteins belonging to relevant functional modules together with network analysis allowed us to deduce that serine-threonine (S-T) phosphorylation modulated protein-protein interactions (PPIs) in a system-wide fashion. We also provide experimental support for this inference by showing that phosphorylation of tubulin on six distinct serine residues abrogated PPIs during the assembly of microtubules. We propose that modulation of PPIs by stimulus-dependent changes in S-T phosphorylation state is a widespread phenomenon applicable to many other signaling systems.\n",
      "Lysine acetylation is a reversible posttranslational modification of proteins and plays a key role in regulating gene expression. Technological limitations have so far prevented a global analysis of lysine acetylation's cellular roles. We used high-resolution mass spectrometry to identify 3600 lysine acetylation sites on 1750 proteins and quantified acetylation changes in response to the deacetylase inhibitors suberoylanilide hydroxamic acid and MS-275. Lysine acetylation preferentially targets large macromolecular complexes involved in diverse cellular processes, such as chromatin remodeling, cell cycle, splicing, nuclear transport, and actin nucleation. Acetylation impaired phosphorylation-dependent interactions of 14-3-3 and regulated the yeast cyclin-dependent kinase Cdc28. Our data demonstrate that the regulatory scope of lysine acetylation is broad and comparable with that of other major posttranslational modifications.\n",
      "Eukaryotic cells replicate by a complex series of evolutionarily conserved events that are tightly regulated at defined stages of the cell division cycle. Progression through this cycle involves a large number of dedicated protein complexes and signaling pathways, and deregulation of this process is implicated in tumorigenesis. We applied high-resolution mass spectrometry-based proteomics to investigate the proteome and phosphoproteome of the human cell cycle on a global scale and quantified 6027 proteins and 20,443 unique phosphorylation sites and their dynamics. Co-regulated proteins and phosphorylation sites were grouped according to their cell cycle kinetics and compared to publicly available messenger RNA microarray data. Most detected phosphorylation sites and more than 20% of all quantified proteins showed substantial regulation, mainly in mitotic cells. Kinase-motif analysis revealed global activation during S phase of the DNA damage response network, which was mediated by phosphorylation by ATM or ATR or DNA-dependent protein kinases. We determined site-specific stoichiometry of more than 5000 sites and found that most of the up-regulated sites phosphorylated by cyclin-dependent kinase 1 (CDK1) or CDK2 were almost fully phosphorylated in mitotic cells. In particular, nuclear proteins and proteins involved in regulating metabolic processes have high phosphorylation site occupancy in mitosis. This suggests that these proteins may be inactivated by phosphorylation in mitotic cells.\n",
      "BACKGROUND: On the basis of large proteomics datasets measured from seven human cell lines we consider their intersection as an approximation of the human central proteome, which is the set of proteins ubiquitously expressed in all human cells. Composition and properties of the central proteome are investigated through bioinformatics analyses.\n",
      "RESULTS: We experimentally identify a central proteome comprising 1,124 proteins that are ubiquitously and abundantly expressed in human cells using state of the art mass spectrometry and protein identification bioinformatics. The main represented functions are proteostasis, primary metabolism and proliferation. We further characterize the central proteome considering gene structures, conservation, interaction networks, pathways, drug targets, and coordination of biological processes. Among other new findings, we show that the central proteome is encoded by exon-rich genes, indicating an increased regulatory flexibility through alternative splicing to adapt to multiple environments, and that the protein interaction network linking the central proteome is very efficient for synchronizing translation with other biological processes. Surprisingly, at least 10% of the central proteome has no or very limited functional annotation.\n",
      "CONCLUSIONS: Our data and analysis provide a new and deeper description of the human central proteome compared to previous results thereby extending and complementing our knowledge of commonly expressed human proteins. All the data are made publicly available to help other researchers who, for instance, need to compare or link focused datasets to a common background.\n",
      "INTERACTIONS:\n",
      "\n",
      "PATHWAYS:\n",
      "\n",
      "<END_CONTEXT>\n",
      "\n",
      "<BEGIN_CONTEXT>ABSTRACTS:\n",
      "The promoter/5' flank sequence, cDNA sequence, and exon/intron structures of the human thrombospondin 3 (THBS3) gene have been determined. THBS3 cDNA clones were obtained by PCR amplification of human fetal lung cDNA using THBS3-specific primers. Analysis of cDNA and genomic sequences showed the THBS3 gene to be composed of 23 exons, 1 more than the number of exons in the previously characterized mouse TSP3 gene. The additional exon results from the division of mouse exon F into exons F1 and F2. The cDNA encodes a polypeptide of 956 amino acids that is highly acidic, with a clustering of acidic side chains in the third quarter of the polypeptide. This region corresponds to seven type III (Ca(2+)-binding) repeats, a feature shared with other thrombospondins. In addition to these type III repeats, four type II (EGF-like) repeats and NH2- and COOH-terminal domains are present in thrombospondin 3. The THBS3 and mouse TSP3 genes differ in intron sizes, but exon sequences and sizes and positions of insertion of introns are conserved to a high degree. The structural organization of the THBS3 gene is of interest because of its close proximity to that of metaxin, with which it shares a common promoter sequence, and to the gene encoding glucocerebrosidase, a deficiency in which causes Gaucher disease.\n",
      "As a base for human transcriptome and functional genomics, we created the \"full-length long Japan\" (FLJ) collection of sequenced human cDNAs. We determined the entire sequence of 21,243 selected clones and found that 14,490 cDNAs (10,897 clusters) were unique to the FLJ collection. About half of them (5,416) seemed to be protein-coding. Of those, 1,999 clusters had not been predicted by computational methods. The distribution of GC content of nonpredicted cDNAs had a peak at approximately 58% compared with a peak at approximately 42%for predicted cDNAs. Thus, there seems to be a slight bias against GC-rich transcripts in current gene prediction procedures. The rest of the cDNAs unique to the FLJ collection (5,481) contained no obvious open reading frames (ORFs) and thus are candidate noncoding RNAs. About one-fourth of them (1,378) showed a clear pattern of splicing. The distribution of GC content of noncoding cDNAs was narrow and had a peak at approximately 42%, relatively low compared with that of protein-coding cDNAs.\n",
      "The reference sequence for each human chromosome provides the framework for understanding genome function, variation and evolution. Here we report the finished sequence and biological annotation of human chromosome 1. Chromosome 1 is gene-dense, with 3,141 genes and 991 pseudogenes, and many coding sequences overlap. Rearrangements and mutations of chromosome 1 are prevalent in cancer and many other diseases. Patterns of sequence variation reveal signals of recent selection in specific genes that may contribute to human fitness, and also in regions where no function is evident. Fine-scale recombination occurs in hotspots of varying intensity along the sequence, and is enriched near genes. These and other studies of human biology and disease encoded within chromosome 1 are made possible with the highly accurate annotated sequence, as part of the completed set of chromosome sequences that comprise the reference human genome.\n",
      "Gaucher disease results from the deficiency of the lysosomal enzyme glucocerebrosidase (EC 3.2.1.45). Although the functional gene for glucocerebrosidase (GBA) and its pseudogene (psGBA), located in close proximity on chromosome 1q21, have been studied extensively, the flanking sequence has not been well characterized. The recent identification of human metaxin (MTX) immediately downstream of psGBA prompted a closer analysis of the sequence of the entire region surrounding the GBA gene. We now report the genomic DNA sequence and organization of a 75-kb region around GBA, including the duplicated region containing GBA and MTX. The origin and endpoints of the duplication leading to the pseudogenes for GBA and MTX are now clearly established. We also have identified three new genes within the 32 kb of sequence upstream to GBA, all of which are transcribed in the same direction as GBA. Of these three genes, the gene most distal to GBA is a protein kinase (clk2). The second gene, propin1, has a 1.5-kb cDNA and shares homology to a rat secretory carrier membrane protein 37 (SCAMP37). Finally, cote1, a gene of unknown function lies most proximal to GBA. The possible contributions of these closely arrayed genes to the more atypical presentations of Gaucher disease is now under investigation.\n",
      "The National Institutes of Health's Mammalian Gene Collection (MGC) project was designed to generate and sequence a publicly accessible cDNA resource containing a complete open reading frame (ORF) for every human and mouse gene. The project initially used a random strategy to select clones from a large number of cDNA libraries from diverse tissues. Candidate clones were chosen based on 5'-EST sequences, and then fully sequenced to high accuracy and analyzed by algorithms developed for this project. Currently, more than 11,000 human and 10,000 mouse genes are represented in MGC by at least one clone with a full ORF. The random selection approach is now reaching a saturation point, and a transition to protocols targeted at the missing transcripts is now required to complete the mouse and human collections. Comparison of the sequence of the MGC clones to reference genome sequences reveals that most cDNA clones are of very high sequence quality, although it is likely that some cDNAs may carry missense variants as a consequence of experimental artifact, such as PCR, cloning, or reverse transcriptase errors. Recently, a rat cDNA component was added to the project, and ongoing frog (Xenopus) and zebrafish (Danio) cDNA projects were expanded to take advantage of the high-throughput MGC pipeline.\n",
      "UNLABELLED: Protein phosphorylation is one of the most common post-translational modifications. It plays key roles in regulating diverse biological processes of liver tissues. To better understand the role of protein phosphorylation in liver functions, it is essential to perform in-depth phosphoproteome analysis of human liver. Here, an enzyme assisted reversed-phase-reversed-phase liquid chromatography (RP-RPLC) approach with both RPLC separations operated with optimized acidic mobile phase was developed. High orthogonal separation was achieved by trypsin digestion of the Glu-C generated peptides in the fractions collected from the first RPLC separation. The phosphoproteome coverage was further improved by using two types of instruments, i.e. TripleTOF 5600 and LTQ Orbitrap Velos. A total of 22,446 phosphorylation sites, corresponding to 6526 nonredundant phosphoproteins were finally identified from normal human liver tissues. Of these sites, 15,229 sites were confidently localized with Ascore13. This dataset was the largest phosphoproteome dataset of human liver. It can be a public resource for the liver research community and holds promise for further biology studies.\n",
      "BIOLOGICAL SIGNIFICANCE: The enzyme assisted approach enabled the two RPLC separations operated both with optimized acidic mobile phases. The identifications from TripleTOF 5600 and Orbitrap Velos are highly complementary. The largest phosphoproteome dataset of human liver was generated.\n",
      "The elucidation of the human genome sequence has made it possible to identify genetic alterations in cancers in unprecedented detail. To begin a systematic analysis of such alterations, we determined the sequence of well-annotated human protein-coding genes in two common tumor types. Analysis of 13,023 genes in 11 breast and 11 colorectal cancers revealed that individual tumors accumulate an average of approximately 90 mutant genes but that only a subset of these contribute to the neoplastic process. Using stringent criteria to delineate this subset, we identified 189 genes (average of 11 per tumor) that were mutated at significant frequency. The vast majority of these genes were not known to be genetically altered in tumors and are predicted to affect a wide range of cellular functions, including transcription, adhesion, and invasion. These data define the genetic landscape of two human cancer types, provide new targets for diagnostic and therapeutic intervention, and open fertile avenues for basic research in tumor biology.\n",
      "INTERACTIONS:\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P49746\", \"intActId\": \"EBI-2530931\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P27797\", \"geneName\": \"CALR\", \"intActId\": \"EBI-1049597\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P49746\", \"intActId\": \"EBI-2530931\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P36957\", \"geneName\": \"DLST\", \"intActId\": \"EBI-351007\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P49746\", \"intActId\": \"EBI-2530931\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q8TDX7\", \"geneName\": \"NEK7\", \"intActId\": \"EBI-1055945\"}, \"numberOfExperiments\": 3, \"organismDiffer\": false}\n",
      "PATHWAYS:\n",
      "\n",
      "<END_CONTEXT>\n",
      "\n",
      "<BEGIN_CONTEXT>ABSTRACTS:\n",
      "We report the remainder of the cDNA sequence for human thrombospondin 2 (hTSP2) including the 5' and 3' untranslated regions. The homology between the two human thrombospondins is highest at the carboxyl-terminal end. The 5' untranslated region of hTSP2 is a GC-rich, 239-bp segment that encodes an in-frame methionine residue embedded in a Kozak consensus sequence. The 3' untranslated region is AT-rich and contains nucleotide sequences implicated in the regulation of mRNA stability. It also contains an 89-bp AT-rich segment with 92% of its nucleotides identical to the same region of the mouse TSP2 3' untranslated region, which suggests an important functional or structural property associated with this region.\n",
      "As a base for human transcriptome and functional genomics, we created the \"full-length long Japan\" (FLJ) collection of sequenced human cDNAs. We determined the entire sequence of 21,243 selected clones and found that 14,490 cDNAs (10,897 clusters) were unique to the FLJ collection. About half of them (5,416) seemed to be protein-coding. Of those, 1,999 clusters had not been predicted by computational methods. The distribution of GC content of nonpredicted cDNAs had a peak at approximately 58% compared with a peak at approximately 42%for predicted cDNAs. Thus, there seems to be a slight bias against GC-rich transcripts in current gene prediction procedures. The rest of the cDNAs unique to the FLJ collection (5,481) contained no obvious open reading frames (ORFs) and thus are candidate noncoding RNAs. About one-fourth of them (1,378) showed a clear pattern of splicing. The distribution of GC content of noncoding cDNAs was narrow and had a peak at approximately 42%, relatively low compared with that of protein-coding cDNAs.\n",
      "Chromosome 6 is a metacentric chromosome that constitutes about 6% of the human genome. The finished sequence comprises 166,880,988 base pairs, representing the largest chromosome sequenced so far. The entire sequence has been subjected to high-quality manual annotation, resulting in the evidence-supported identification of 1,557 genes and 633 pseudogenes. Here we report that at least 96% of the protein-coding genes have been identified, as assessed by multi-species comparative sequence analysis, and provide evidence for the presence of further, otherwise unsupported exons/genes. Among these are genes directly implicated in cancer, schizophrenia, autoimmunity and many other diseases. Chromosome 6 harbours the largest transfer RNA gene cluster in the genome; we show that this cluster co-localizes with a region of high transcriptional activity. Within the essential immune loci of the major histocompatibility complex, we find HLA-B to be the most polymorphic gene on chromosome 6 and in the human genome.\n",
      "The National Institutes of Health's Mammalian Gene Collection (MGC) project was designed to generate and sequence a publicly accessible cDNA resource containing a complete open reading frame (ORF) for every human and mouse gene. The project initially used a random strategy to select clones from a large number of cDNA libraries from diverse tissues. Candidate clones were chosen based on 5'-EST sequences, and then fully sequenced to high accuracy and analyzed by algorithms developed for this project. Currently, more than 11,000 human and 10,000 mouse genes are represented in MGC by at least one clone with a full ORF. The random selection approach is now reaching a saturation point, and a transition to protocols targeted at the missing transcripts is now required to complete the mouse and human collections. Comparison of the sequence of the MGC clones to reference genome sequences reveals that most cDNA clones are of very high sequence quality, although it is likely that some cDNAs may carry missense variants as a consequence of experimental artifact, such as PCR, cloning, or reverse transcriptase errors. Recently, a rat cDNA component was added to the project, and ongoing frog (Xenopus) and zebrafish (Danio) cDNA projects were expanded to take advantage of the high-throughput MGC pipeline.\n",
      "A novel form of human thrombospondin was identified during the screening of a human fibroblast cDNA library. We report the cDNA sequence for 1.8 kb of the 3' end of the cDNA, plus an additional 937 bp of 3'-untranslated sequence. The translated sequence reveals a high degree of similarity to thrombospondin I. The homology ranges from 56 to 80% for different regions within the two proteins. The repeating segments of amino acid sequence identified in thrombospondin I were found to be conserved in thrombospondin II. The new form of thrombospondin hybridizes to a 7.5-kb message by Northern analysis. The THBS2 gene is located at the distal long arm of chromosome 6 at 6q27. The gene is transcribed in fibroblasts, smooth muscle cells, and an osteosarcoma cell line, at levels somewhat lower than that of thrombospondin I. Umbilical vein endothelial cells do not transcribe thrombospondin II under the conditions of this study. These findings suggest that previous studies of thrombospondin function need to be reassessed to identify the functions specific to each molecule.\n",
      "To clarify the role of CD36 as a TSP receptor and to investigate the mechanisms of the TSP-CD36 interaction, transfection studies were performed using CD36-cDNA in a CDM8 plasmid. Jurkat cells transfected with CD36 cDNA express an 88kD membrane surface protein and acquire the ability to bind thrombospondin. The TSP amino acid sequence, CSVTCG, mediates the interaction of thrombospondin with CD36. CD36 transfectants but not control transfectants bind radiolabeled tyrosinated peptide (YCSVTCG). The hexapeptide inhibits thrombospondin expression on activated human platelets and results in diminished platelet aggregation. CSVTCG-albumin conjugates support CD36-dependent adhesion of tumor cells. We conclude that the CSVTCG repeat sequence is a crucial determinant of CD36 thrombospondin binding.\n",
      "The thrombospondin (TSP) family of extracellular glycoproteins consists of five members in vertebrates, TSP1 to -4 and TSP5/cartilage oligomeric matrix protein, and a single member in Drosophila. TSPs are modular multimeric proteins. The C-terminal end of a monomer consists of 3-6 EGF-like modules; seven tandem 23-, 36-, or 38-residue aspartate-rich, Ca(2+)-binding repeats; and an approximately 230-residue C-terminal sequence. The Ca(2+)-binding repeats and C-terminal sequence are spaced almost exactly the same in different TSPs and share many blocks of identical residues. We studied the C-terminal portion of human TSP2 from the third EGF-like module through the end of the protein (E3CaG2). E3CaG2, CaG2 lacking the EGF module, and Ca2 composed of only the Ca(2+)-binding repeats were expressed using recombinant baculoviruses and purified from conditioned media of insect cells. As previously described for intact TSP1, E3CaG2 bound Ca(2+) in a cooperative manner as assessed by equilibrium dialysis, and its circular dichroism spectrum was sensitive to the presence of Ca(2+). Mass spectrometry of the recombinant proteins digested with endoproteinase Asp-N revealed that disulfide pairing of the 18 cysteines in the Ca(2+)-binding repeats and C-terminal sequence is sequential, i.e. a 1-2, 3-4, 5-6, etc., pattern.\n",
      "Lumbar-disc herniation (LDH), one of the most common musculoskeletal diseases, has strong genetic determinants. Recently, several genes that encode extracellular matrix (ECM) proteins in the intervertebral disc have been reported to associate with LDH. Thrombospondins (THBSs) 1 and 2 are good candidates for the LDH susceptibility gene: They are intervertebral disc ECM proteins that regulate the effective levels of matrix metalloproteinases (MMPs) 2 and 9, which are key effectors of ECM remodeling. Here, we report that THBS2 is associated with LDH in Japanese populations. An intronic SNP in THBS2 (IVS10-8C --> T; rs9406328) showed significant association (p = 0.0000028) with LDH in two independent Japanese populations. This SNP, located in a polypyrimidine tract upstream of the 3' splice site of intron 10, exerts allelic differences on exon 11 skipping rates in vivo, with the susceptibility allele showing increased skipping. Skipping of exon 11 results in decreased THBS2 interaction with MMP2 and MMP9. Further, a missense SNP in MMP9 (Q279R; rs17576) is also strongly associated with LDH in the Japanese population (p = 0.00049) and shows a combinatorial effect with THBS2 (odds ratio 3.03, 95% confidence interval 1.58-5.77). Thus, a splicing-affecting SNP in THBS2 and a missense SNP in MMP9 are associated with susceptibility to LDH. Our data indicate that regulation of intervertebral disc ECM metabolism by the THBS2-MMP system plays an essential role in the etiology and pathogenesis of LDH.\n",
      "Thus far the clinical benefits seen in breast cancer patients treated with drugs targeting the vascular endothelial growth factor (VEGF) pathway are only modest. Consequently, additional antiangiogenic approaches for treatment of breast cancer need to be investigated. Thrombospondin-2 (TSP-2) has been shown to inhibit tumor growth and angiogenesis with a greater potency than the related molecule TSP-1. The systemic effects of TSP-2 on tumor metastasis and the underlying molecular mechanisms of the antiangiogenic activity of TSP-2 have remained poorly understood. We generated a recombinant fusion protein consisting of the N-terminal region of TSP-2 and the IgG-Fc1 fragment (N-TSP2-Fc) and could demonstrate that the antiangiogenic activity of N-TSP2-Fc is dependent on the CD36 receptor. We found that N-TSP2-Fc inhibited VEGF-induced tube formation of human dermal microvascular endothelial cells (HDMEC) on matrigel in vitro and that concurrent incubation of anti-CD36 antibody with N-TSP2-Fc resulted in tube formation that was comparable to untreated control. N-TSP2-Fc potently induced apoptosis of HDMEC in vitro in a CD36-dependent manner. Moreover, we could demonstrate a CD36 receptor-mediated loss of mitochondrial membrane potential and activation of caspase-3 in HDMEC in vitro. Daily intraperitoneal injections of N-TSP2-Fc resulted in a significant inhibition of the growth of human MDA-MB-435 and MDA-MB-231 tumor cells grown in the mammary gland of immunodeficient nude mice and in reduced tumor vascularization. Finally, increased serum concentrations of N-TSP2-Fc significantly inhibited regional metastasis to lymph nodes and distant metastasis to lung as shown by quantitative real-time alu PCR. These results identify N-TSP2-Fc as a potent systemic inhibitor of tumor metastasis and provide strong evidence for an important role of the CD36 receptor in mediating the antiangiogenic activity of TSP-2.\n",
      "Thrombospondins (THBSs) are secreted glycoproteins that have key roles in interactions between cells and the extracellular matrix. Here, we describe the 2.6-A-resolution crystal structure of the glycosylated signature domain of human THBS2, which includes three epidermal growth factor-like modules, 13 aspartate-rich repeats and a lectin-like module. These elements interact extensively to form three structural regions termed the stalk, wire and globe. The THBS2 signature domain is stabilized by these interactions and by a network of 30 bound Ca(2+) ions and 18 disulfide bonds. The structure suggests how genetic alterations of THBSs result in disease.\n",
      "Thrombospondins (THBSs) are multimodular, secreted proteins characterized by a signature domain comprising a unique set of 13 calcium-binding repeats flanked by epidermal growth factor (EGF)-like and lectin-like modules. A polymorphism that changes a conserved Asn to Ser at residue 700 in the most N-terminal calcium-binding repeat of THBS-1 (repeat 1C) is found in 8-10% of European populations and has been linked to increased risk of premature coronary artery disease. The Ser substitution leads to altered stability in the EGF-like and wire modules of the THBS-1 signature domain as assessed by differential scanning calorimetry carried out in 2 mm or 200 mum calcium. Studies of the melting profiles of the THBS-2 signature domain proteins with Asn or Ser at position 702 (homologous to 700 in THBS-1) revealed that the impact of the Ser allele is similar in both THBS-1 and THBS-2. Structure determination of the Ser(702) THBS-2 variant in 2 mm calcium showed that repeat 1C contains two bound calcium ions as in the crystal of the Asn(702) protein, including the ion that is coordinated by Asn(702), and is associated with changes in conformation of repeat 1C and the adjacent EGF-like modules. The Ser substitution leads to the decreased ability of soluble THBS-2 signature domain protein to bind 4B6.13, a conformation-sensitive monoclonal antibody that recognizes an epitope in repeat 1C. These results indicate that although THBS harboring the Ser allele binds a full complement of calcium ions, repeat 1C is altered, leading to destabilization of surrounding structures.\n",
      "INTERACTIONS:\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"P35442\", \"intActId\": \"EBI-2466249\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P54290\", \"geneName\": \"Cacna2d1\", \"intActId\": \"EBI-2466294\"}, \"numberOfExperiments\": 2, \"organismDiffer\": true}\n",
      "PATHWAYS:\n",
      "\n",
      "<END_CONTEXT>\n",
      "\n",
      "<BEGIN_CONTEXT>ABSTRACTS:\n",
      "Two cDNA clones encoding two major human trypsinogen isozymes were isolated from a human pancreatic cDNA library. The deduced amino acid (aa) sequences of the two trypsinogen precursors are found to have 89% sequence homology, and have the same number of aa (247), including 15 aa for a signal peptide and 8 aa for an activation peptide. Southern blot analysis of human genomic DNA using the cloned cDNA as a probe, revealed that the human trypsinogen genes constitute a multigene family of more than ten genes.\n",
      "The human beta T cell receptor (TCR) locus, comprising a complex family of genes, has been sequenced. The locus contains two types of coding elements--TCR elements (65 variable gene segments and two clusters of diversity, joining, and constant segments) and eight trypsinogen genes --that constitute 4.6 percent of the DNA. Genome-wide interspersed repeats and locus-specific repeats span 30 and 47 percent, respectively, of the 685-kilobase sequence. A comparison of the germline variable elements with their approximately 300 complementary DNA counterparts reveals marked differential patterns of variable gene expression, the importance of exonuclease activity in generating TCR diversity, and the predominant tendency for only functional variable elements to be present in complementary DNA libraries.\n",
      "As a base for human transcriptome and functional genomics, we created the \"full-length long Japan\" (FLJ) collection of sequenced human cDNAs. We determined the entire sequence of 21,243 selected clones and found that 14,490 cDNAs (10,897 clusters) were unique to the FLJ collection. About half of them (5,416) seemed to be protein-coding. Of those, 1,999 clusters had not been predicted by computational methods. The distribution of GC content of nonpredicted cDNAs had a peak at approximately 58% compared with a peak at approximately 42%for predicted cDNAs. Thus, there seems to be a slight bias against GC-rich transcripts in current gene prediction procedures. The rest of the cDNAs unique to the FLJ collection (5,481) contained no obvious open reading frames (ORFs) and thus are candidate noncoding RNAs. About one-fourth of them (1,378) showed a clear pattern of splicing. The distribution of GC content of noncoding cDNAs was narrow and had a peak at approximately 42%, relatively low compared with that of protein-coding cDNAs.\n",
      "Human chromosome 7 has historically received prominent attention in the human genetics community, primarily related to the search for the cystic fibrosis gene and the frequent cytogenetic changes associated with various forms of cancer. Here we present more than 153 million base pairs representing 99.4% of the euchromatic sequence of chromosome 7, the first metacentric chromosome completed so far. The sequence has excellent concordance with previously established physical and genetic maps, and it exhibits an unusual amount of segmentally duplicated sequence (8.2%), with marked differences between the two arms. Our initial analyses have identified 1,150 protein-coding genes, 605 of which have been confirmed by complementary DNA sequences, and an additional 941 pseudogenes. Of genes confirmed by transcript sequences, some are polymorphic for mutations that disrupt the reading frame.\n",
      "DNA sequence and annotation of the entire human chromosome 7, encompassing nearly 158 million nucleotides of DNA and 1917 gene structures, are presented. To generate a higher order description, additional structural features such as imprinted genes, fragile sites, and segmental duplications were integrated at the level of the DNA sequence with medical genetic data, including 440 chromosome rearrangement breakpoints associated with disease. This approach enabled the discovery of candidate genes for developmental diseases including autism.\n",
      "The National Institutes of Health's Mammalian Gene Collection (MGC) project was designed to generate and sequence a publicly accessible cDNA resource containing a complete open reading frame (ORF) for every human and mouse gene. The project initially used a random strategy to select clones from a large number of cDNA libraries from diverse tissues. Candidate clones were chosen based on 5'-EST sequences, and then fully sequenced to high accuracy and analyzed by algorithms developed for this project. Currently, more than 11,000 human and 10,000 mouse genes are represented in MGC by at least one clone with a full ORF. The random selection approach is now reaching a saturation point, and a transition to protocols targeted at the missing transcripts is now required to complete the mouse and human collections. Comparison of the sequence of the MGC clones to reference genome sequences reveals that most cDNA clones are of very high sequence quality, although it is likely that some cDNAs may carry missense variants as a consequence of experimental artifact, such as PCR, cloning, or reverse transcriptase errors. Recently, a rat cDNA component was added to the project, and ongoing frog (Xenopus) and zebrafish (Danio) cDNA projects were expanded to take advantage of the high-throughput MGC pipeline.\n",
      "BACKGROUND & AIMS: Mutations of the cationic trypsinogen have been described in hereditary pancreatitis. We report a new trypsinogen mutation in the activation peptide of the proenzyme in a family with chronic pancreatitis.\n",
      "METHODS: The coding region of the cationic trypsinogen gene was sequenced after polymerase chain reaction amplification. The following peptides homologous to the N-terminal end of cationic trypsinogen were synthesized (one-letter code, mutated amino acid underlined): wild-type peptide, APFDDDDKIVGG; pD22G, APFDDDGKIVGG; pK23R, APFDDDDRIVGG. The sequences of pD22G and pK23R correspond to the recently identified mutation K23R and to the mutation described here (D22G). To mimic trypsinogen activation, these peptides were digested with trypsin for 30 minutes at pH 5.0-8. 0, and the fragments were analyzed by high-performance liquid chromatography.\n",
      "RESULTS: In a family with clinical evidence of hereditary chronic pancreatitis, a missense mutation of codon 22 (GAC-->GGC) of the cationic trypsinogen was found. This mutation results in a substitution of aspartic acid by glycine; therefore, the mutation was called D22G. Chromatographic analysis of tryptic digests of the peptides pD22G and pK23R showed hydrolysis rates of 22% and 75%, respectively, whereas the wild-type peptide was hydrolyzed at only 6%. The cleavage rates were reduced at lower pH, and no hydrolysis occurred without trypsin.\n",
      "CONCLUSIONS: The activation peptides of the trypsinogen variants D22G and K23R could be released at a higher rate than in wild-type trypsinogen, resulting in increased amounts of trypsin in the pancreas, which could initiate pancreatitis.\n",
      "It has previously been reported that two kinds of human gastric adenocarcinoma cell lines (STKM-1 and MKN28) secrete a trypsin-like enzyme. In this study, four molecular forms of the enzyme (26, 25, 24 and 23 kDa on non-reducing SDS/PAGE) were purified from the serum-free conditioned medium of STKM-1 cells. Analysis of N-terminal amino acid sequences showed that the 26 kDa protein was a two-chain form of trypsinogen 1 which had been produced by proteolytic cleavage of the Arg107-Val108 bond of trypsinogen 1, and the 24 kDa protein was the one-chain form of trypsinogen 1. The 25 and 23 kDa proteins were the activated forms of the two-chain and one-chain trypsinogen 1 respectively. Isoelectric focusing gave pI values of 6.3 and 6.6 for the 26 kDa two-chain form and the 24 kDa one-chain form of trypsinogen 1 respectively. Comparison of the proteolytic activities indicated that the one-chain trypsin 1 had amidolytic activity about four times higher than that of the two-chain enzyme.\n",
      "A simple method for the purification of anionic and cationic trypsinogen and trypsin from human pancreatic juice applying affinity chromatography on aprotinin coupled Sepharose is described together with the N-terminal amino acid sequences for both trypsinogens. In addition, enzyme-linked immunoabsorbent assay (ELISA) methods for the determination of anionic and cationic trypsin-like immunoreactivity (irAT and irCT) are described. Normal serum levels are 21.3 +/- 7.4 micrograms/l and 27.8 +/- 9.0 microgram/l for irAT and irCT respectively and the accuracy of these assays is 6-10%. In our population, the normal ratio between irCT and irAT in serum is 1.36 +/- 0.42. In normal serum trypsin-like immunoreactivity consists solely of trypsinogen. In acute pancreatitis there is an increase over normal of both irAT and irCT with a proportionally greater increase in irAT than irCT. Similar changes are also found in uremic patients.\n",
      "Hereditary pancreatitis (HP) is a rare, early-onset genetic disorder characterized by epigastric pain and often more serious complications. We now report that an Arg-His substitution at residue 117 of the cationic trypsinogen gene is associated with the HP phenotype. This mutation was observed in all HP affected individuals and obligate carriers from five kindreds, but not in individuals who married into the families nor in 140 unrelated individuals. X-ray crystal structure analysis, molecular modelling, and protein digest data indicate that the Arg 117 residue is a trypsin-sensitive site. Cleavage at this site is probably part of a fail-safe mechanism by which trypsin, which is activated within the pancreas, may be inactivated; loss of this cleavage site would permit autodigestion resulting in pancreatitis.\n",
      "OBJECTIVES: Mutations of the cationic trypsinogen (CT) and the serine protease inhibitor, Kazal type 1 (SPINK 1) are associated with chronic pancreatitis. After mutational screening of a cohort of patients with nonalcoholic chronic pancreatitis, we report three novel variants of the trypsinogen molecule and the clinical characteristics of the carriers.\n",
      "METHODS: The coding region of the exon 2 and 3 of the CT gene of 523 patients with chronic nonalcoholic pancreatitis (108 patients with suspected hereditary pancreatitis (HP) and 415 patients with \"idio pathic\" pancreatitis [IP]) and 82 controls was analyzed after polymerase chain reaction amplification. Clinical characteristics were obtained by questioning the patients and their relatives and physicians. HP was suspected when two members of a family had chronic pancreatitis. A restriction digestion was used to analyze the N34S mutation SPINK1.\n",
      "RESULTS: The mutation R122H of the cationic trypsinogen was found in 21 index patients, N291 in six index patients, and A16V and D22G in one index patient, all from HP families. The N34S mutation of SPINK1 was found in two index patients with a family history of HP. In three patients, the novel point mutations L104P, R116C, and C139F of the cationic trypsinogen were found. A clear autosomally dominant inheritance of chronic pancreatitis was not present in these families. In 75 index patients from HP families (69.4%), no mutation could be found. The SPINK 1-mutation N34S was detected in only one patient carrying a CT mutation, and was found in 68 (16.4%) of patients with IP.\n",
      "CONCLUSIONS: The R122H and N291 mutations of CT are the most common disease-associated mutations in HP; the N34S mutation of SPINK I is the most frequent genetic risk factor associated with IP. The CT gene carries several variations that could be associated with chronic pancreatitis. To avoid overestimating the pathogenetic impact of novel trypsinogen variants, a detailed clinical characterization of all patients with early onset chronic pancreatitis is mandatory.\n",
      "The crystal structure of human pancreatic cationic trypsin showed the chemical modification of Tyr154, which was originally described as phosphorylation [Gaboriaud C, Serre L, Guy-Crotte O, Forest E & Fontecilla-Camps JC (1996) J Mol Biol259, 995-1010]. Here we report that Tyr154 is sulfated, not phosphorylated. Cationic and anionic trypsinogens were purified from human pancreatic juice and subjected to alkaline hydrolysis. Modified tyrosine amino acids were separated on a Dowex cation-exchange column and analyzed by thin layer chromatography. Both human cationic and anionic trypsinogens contained tyrosine sulfate, but no tyrosine phosphate, whereas bovine trypsinogen contained neither. Furthermore, incorporation of [(35)S]SO(4) into human cationic trypsinogen transiently expressed by human embryonic kidney 239T cells was demonstrated. Mutation of Tyr154 to Phe abolished radioactive sulfate incorporation, confirming that Tyr154 is the site of sulfation in cationic trypsinogen. Sulfated pancreatic cationic trypsinogen exhibited faster autoactivation than a nonsulfated recombinant form, suggesting that tyrosine sulfation of trypsinogens might enhance intestinal digestive zymogen activation in humans. Finally, sequence alignment revealed that the sulfation motif is only conserved in primate trypsinogens, suggesting that typsinogen sulfation is absent in other vertebrates.\n",
      "Human cationic and anionic trypsins are sulfated on Tyr154, a residue which helps to shape the prime side substrate-binding subsites. Here, we used phage display technology to assess the significance of tyrosine sulfation for the specificity of human trypsins. The prime side residues P1'-P4' in the binding loop of bovine pancreatic trypsin inhibitor (BPTI) were fully randomized and tight binding inhibitor phages were selected against non-sulfated and sulfated human cationic trypsin. The selection pattern for the two targets differed mostly at the P2' position, where variants selected against non-sulfated trypsin contained primarily aliphatic residues (Leu, Ile, Met), while variants selected against sulfated trypsin were enriched also for Arg. BPTI variants carrying Arg, Lys, Ile, Leu or Ala at the P2' position of the binding loop were purified and equilibrium dissociation constants were determined against non-sulfated and sulfated cationic and anionic human trypsins. BPTI variants harboring apolar residues at P2' exhibited 3-12-fold lower affinity to sulfated trypsin relative to the non-sulfated enzyme, whereas BPTI variants containing basic residues at P2' had comparable affinity to both trypsin forms. Taken together, the observations demonstrate that the tyrosyl sulfate in human trypsins interacts with the P2' position of the substrate-like inhibitor and this modification increases P2' selectivity towards basic side chains.\n",
      "The serpins have evolved to be the predominant family of serine-protease inhibitors in man. Their unique mechanism of inhibition involves a profound change in conformation, although the nature and significance of this change has been controversial. Here we report the crystallographic structure of a typical serpin-protease complex and show the mechanism of inhibition. The conformational change is initiated by reaction of the active serine of the protease with the reactive centre of the serpin. This cleaves the reactive centre, which then moves 71 A to the opposite pole of the serpin, taking the tethered protease with it. The tight linkage of the two molecules and resulting overlap of their structures does not affect the hyperstable serpin, but causes a surprising 37% loss of structure in the protease. This is induced by the plucking of the serine from its active site, together with breakage of interactions formed during zymogen activation. The disruption of the catalytic site prevents the release of the protease from the complex, and the structural disorder allows its proteolytic destruction. It is this ability of the conformational mechanism to crush as well as inhibit proteases that provides the serpins with their selective advantage.\n",
      "INTERACTIONS:\n",
      "\n",
      "PATHWAYS:\n",
      "\n",
      "<END_CONTEXT>\n",
      "\n",
      "<BEGIN_CONTEXT>ABSTRACTS:\n",
      "Thrombospondin-1 is an adhesive glycoprotein that is involved in cellular attachment, spreading, migration, and proliferation. To date, four genes have been identified that encode for the members of the thrombospondin gene family. These four genes are homologous to each other in the EGF-like (type 2) repeats, the calcium-binding (type 3) motifs, and the COOH-terminal. The latter has been reported to be a cell-binding domain in thrombospondin-1. Phylogenetic trees have been constructed from the multisequence alignment of thrombospondin sequences from human, mouse, chicken, and frog. Two different algorithms generate comparable results in terms of the topology and the branch lengths. The analysis indicates that an early form of the thrombospondin gene duplicated about 925 million years ago. The gene duplication that produced the thrombospondin-1 and -2 branches of the family is predicted to have occurred 583 million years ago, whereas the gene duplication that produced the thrombospondin-3 and -4 branches of the family is predicted to have occurred 644 million years ago. These results indicate that the members of the thrombospondin gene family have existed throughout the evolution of the animal kingdom and thus probably participate in functions that are common to most of its members.\n",
      "The thrombospondins are a family of extracellular calcium binding proteins that are involved in cell proliferation, adhesion, and migration. We have sequenced full-length human thrombospondin-4 and characterized the recombinant protein. In contrast to Xenopus laevis thrombospondin-4, the human protein contains an RGD cell binding sequence in the third type 3 repeat. Transfection of mouse NIH3T3 fibroblasts or C2C12 myoblasts with a full-length human thrombospondin-4 cDNA results in the expression of a polypeptide with a reduced molecular weight of 140,000. In the absence of reducing agent, the expressed protein has an apparent molecular weight of 550,000. Recombinant thrombospondin-4 has been purified from the culture supernatant by heparin-Sepharose and anti-thrombospondin-4 antibody-Affi-gel affinity chromatography. Electron microscopy indicates that thrombospondin-4 is composed of five subunits with globular domains at each end. The observation of a calcium-dependent change in the electron microscopic appearance of thrombospondin-4 is consistent with limited tryptic digestion data that indicate that thrombospondin-4 is resistant to digestion in the presence of calcium. These data indicate that thrombospondin-4 is a pentameric protein that binds to heparin and calcium.\n",
      "As a base for human transcriptome and functional genomics, we created the \"full-length long Japan\" (FLJ) collection of sequenced human cDNAs. We determined the entire sequence of 21,243 selected clones and found that 14,490 cDNAs (10,897 clusters) were unique to the FLJ collection. About half of them (5,416) seemed to be protein-coding. Of those, 1,999 clusters had not been predicted by computational methods. The distribution of GC content of nonpredicted cDNAs had a peak at approximately 58% compared with a peak at approximately 42%for predicted cDNAs. Thus, there seems to be a slight bias against GC-rich transcripts in current gene prediction procedures. The rest of the cDNAs unique to the FLJ collection (5,481) contained no obvious open reading frames (ORFs) and thus are candidate noncoding RNAs. About one-fourth of them (1,378) showed a clear pattern of splicing. The distribution of GC content of noncoding cDNAs was narrow and had a peak at approximately 42%, relatively low compared with that of protein-coding cDNAs.\n",
      "The National Institutes of Health's Mammalian Gene Collection (MGC) project was designed to generate and sequence a publicly accessible cDNA resource containing a complete open reading frame (ORF) for every human and mouse gene. The project initially used a random strategy to select clones from a large number of cDNA libraries from diverse tissues. Candidate clones were chosen based on 5'-EST sequences, and then fully sequenced to high accuracy and analyzed by algorithms developed for this project. Currently, more than 11,000 human and 10,000 mouse genes are represented in MGC by at least one clone with a full ORF. The random selection approach is now reaching a saturation point, and a transition to protocols targeted at the missing transcripts is now required to complete the mouse and human collections. Comparison of the sequence of the MGC clones to reference genome sequences reveals that most cDNA clones are of very high sequence quality, although it is likely that some cDNAs may carry missense variants as a consequence of experimental artifact, such as PCR, cloning, or reverse transcriptase errors. Recently, a rat cDNA component was added to the project, and ongoing frog (Xenopus) and zebrafish (Danio) cDNA projects were expanded to take advantage of the high-throughput MGC pipeline.\n",
      "The matrix protein thrombospondin-4 has an acidic amphipathic C-terminal peptide (C21) which stimulates erythroid cell proliferation. Here we show that C21 stimulates red cell formation in anemic mice in vivo. In vitro experiments indicated that the peptide-mediated increase of erythroid colony formation in cultures of human CD34+ hematopoietic progenitor cells was possible only under continuous presence of erythropoietin. In the absence of this cytokine, C21 stimulated exclusively myeloid colony formation. Therefore, the peptide is not a specific erythroid differentiation factor. In fact, it is mitogenic in non-erythroid cells, such as skin fibroblasts and kidney epithelial cells. In erythroleukemic TF-1 cells, it actually decreased the production of the erythroid differentiation marker glycophorin A. C21-affinity chromatography revealed regulator of differentiation 1 (ROD1) as a major C21-binding protein. ROD1 is the hematopoietic cell paralog of polypyrimidine tract binding proteins (PTBs), RNA splice regulators which regulate differentiation by repressing tissue-specific exons. ROD1 binding to C21 was strongly inhibited by synthetic RNAs in the order poly A > poly U > poly G = poly C and was weakly inhibited by a synthetic phosphorylated peptide mimicking the C-terminal domain of RNA polymerase II. Cellular overexpression or knockdown experiments of ROD1 suggest a role for this protein in the mitogenic activity of C21. Since the nuclear proteins ROD1 and PTBs regulate differentiation at a posttranscriptional level and there is a fast nuclear uptake of C21, we put forward the idea that the peptide is internalized, goes to the nucleus and maintains cells in a proliferative state by supporting ROD1-mediated inhibition of differentiation.\n",
      "Thrombospondin (Thbs) proteins are induced in sites of tissue damage or active remodeling. The endoplasmic reticulum (ER) stress response is also prominently induced with disease where it regulates protein production and resolution of misfolded proteins. Here we describe a function for Thbs as ER-resident effectors of an adaptive ER stress response. Thbs4 cardiac-specific transgenic mice were protected from myocardial injury, whereas Thbs4(-/-) mice were sensitized to cardiac maladaptation. Thbs induction produced a unique profile of adaptive ER stress response factors and expansion of the ER and downstream vesicles. Thbs bind the ER lumenal domain of activating transcription factor 6 (Atf6) to promote its nuclear shuttling. Thbs4(-/-) mice showed blunted activation of Atf6 and other ER stress-response factors with injury, and Thbs4-mediated protection was lost upon Atf6 deletion. Hence, Thbs can function inside the cell during disease remodeling to augment ER function and protect through a mechanism involving regulation of Atf6.\n",
      "Thrombospondins are secreted, extracellular matrix (ECM) proteins that are upregulated in the heart and other tissues in response to ischemic injury and myocardial stress. Roles for thrombospondins after they are secreted have been examined in a variety of disease models, including myocardial pathology. However, a recent study published in the journal Cell by Lynch et al shifts this paradigm by focusing on roles for intracellular thrombospondins; these authors showed that thrombospondin-4 (Thbs4) can function from within cells to protect the heart by enhancing adaptive aspects of the endoplasmic reticulum (ER) stress response that are mediated by activating transcription factor 6, ATF6. Although this study was carried out in the cardiac context, the results add to our understanding of protein folding and quality control in all tissues. Moreover, the findings underscore the potential widespread therapeutic benefit of enhancing adaptive responses that are regulated by ATF6.\n",
      "PURPOSE OF REVIEW: Thrombospondins (TSPs) are secreted extracellular matrix (ECM) proteins from TSP family, which consists of five homologous members. They share a complex domain structure and have numerous binding partners in ECM and multiple cell surface receptors. Information that has emerged over the past decade identifies TSPs as important mediators of cellular homeostasis, assigning new important roles in cardiovascular pathology to these proteins.\n",
      "RECENT FINDINGS: Recent studies of the functions of TSP in the cardiovascular system, diabetes and aging, which placed several TSPs in a position of critical regulators, demonstrated the involvement of these proteins in practically every aspect of cardiovascular pathophysiology related to atherosclerosis: inflammation, immunity, leukocyte recruitment and function, function of vascular cells, angiogenesis, and responses to hypoxia, ischemia and hyperglycemia. TSPs are also critically important in the development and ultimate outcome of the complications associated with atherosclerosis--myocardial infarction, and heart hypertrophy and failure. Their expression and significance increase with age and with the progression of diabetes, two major contributors to the development of atherosclerosis and its complications.\n",
      "SUMMARY: This overview of recent literature examines the latest information on the newfound functions of TSPs that emphasize the importance of ECM in cardiovascular homeostasis and pathology. The functions of TSPs in myocardium, vasculature, vascular complications of diabetes, aging and immunity are discussed.\n",
      "Few studies are available in human populations investigating involvement of vascular inflammation and oxidative stress-related dysfunctional transformation of high-density lipoprotein (HDL) in establishing cardiovascular disease (CVD) risk. To this end, the current work investigated a subgroup of post-infarction patients at high-risk for recurrent events defined by high levels of HDL cholesterol (HDL-C) and concurrently high levels of C-reactive protein (CRP). Thrombospondin-4 (TSP-4), a matricellular protein of vessel walls associated with inflammation, was investigated in terms of CVD risk using multivariable modelling with a well-characterised functional genetic polymorphism of THBS4 (A387P, rs1866389) along with previously demonstrated risk-related functional genetic polymorphisms of CYBA (C242T, rs4673) and CETP (TaqIB, rs708272), and a set of blood markers. Results revealed risk-association for the gain-of-function P-allele of the THBS4 polymorphism (hazard ratio 2.00, 95% confidence interval 1.10-3.65, p=0.024). Additionally, von Willebrand factor was associated with D-dimer levels in the higher-risk P allele patients suggestive of a connection between endothelial dysfunction and thrombogenesis. In conclusion, TSP-4, a matricellular protein involved in regulating vascular inflammation, plays a role in establishing recurrent coronary risk in post-infarction patients with high levels of HDL-C and CRP. Further studies should focus on additional effects of vascular inflammatory processes on anti-atherogenic functionality of HDL particles.\n",
      "INTERACTIONS:\n",
      "\n",
      "PATHWAYS:\n",
      "\n",
      "<END_CONTEXT>\n",
      "\n",
      "<BEGIN_CONTEXT>ABSTRACTS:\n",
      "Sodium butyrate is a multifunctional agent known to inhibit cell proliferation and to induce differentiation by modulating transcription. We have performed differential display analysis to identify transcriptional targets of sodium butyrate in Balb/c BP-A31 mouse fibroblasts. A novel butyrate-induced transcript B-ind1 has been cloned by this approach. The human homologue of this transcript contains an open reading frame that codes for a protein of 370 amino acids without known functional motifs. In transfected cells, the B-ind1 protein has been found to potentiate different effects of the small GTPase Rac1, such as c-Jun N-terminal kinase activation and transcriptional activity of nuclear factor kappaB (NF-kappaB). In addition, we have demonstrated that B-ind1 forms complexes with the constitutively activated Rac1 protein. To investigate the role of B-ind1 in Rac1 signaling, we have constructed several deletion mutants of B-ind1 and tested their ability to affect the activation of NF-kappaB by Rac1. Interestingly, the fragment encoding the median region of human B-ind1 acted as a dominant-negative variant to block Rac1-mediated NF-kappaB activity. These data define B-ind1 as a novel component of Rac1-signaling pathways leading to the modulation of gene expression.\n",
      "Three hundred cDNAs containing putatively entire open reading frames (ORFs) for previously undefined genes were obtained from CD34+ hematopoietic stem/progenitor cells (HSPCs), based on EST cataloging, clone sequencing, in silico cloning, and rapid amplification of cDNA ends (RACE). The cDNA sizes ranged from 360 to 3496 bp and their ORFs coded for peptides of 58-752 amino acids. Public database search indicated that 225 cDNAs exhibited sequence similarities to genes identified across a variety of species. Homology analysis led to the recognition of 50 basic structural motifs/domains among these cDNAs. Genomic exon-intron organization could be established in 243 genes by integration of cDNA data with genome sequence information. Interestingly, a new gene named as HSPC070 on 3p was found to share a sequence of 105bp in 3' UTR with RAF gene in reversed transcription orientation. Chromosomal localizations were obtained using electronic mapping for 192 genes and with radiation hybrid (RH) for 38 genes. Macroarray technique was applied to screen the gene expression patterns in five hematopoietic cell lines (NB4, HL60, U937, K562, and Jurkat) and a number of genes with differential expression were found. The resource work has provided a wide range of information useful not only for expression genomics and annotation of genomic DNA sequence, but also for further research on the function of genes involved in hematopoietic development and differentiation.\n",
      "As a base for human transcriptome and functional genomics, we created the \"full-length long Japan\" (FLJ) collection of sequenced human cDNAs. We determined the entire sequence of 21,243 selected clones and found that 14,490 cDNAs (10,897 clusters) were unique to the FLJ collection. About half of them (5,416) seemed to be protein-coding. Of those, 1,999 clusters had not been predicted by computational methods. The distribution of GC content of nonpredicted cDNAs had a peak at approximately 58% compared with a peak at approximately 42%for predicted cDNAs. Thus, there seems to be a slight bias against GC-rich transcripts in current gene prediction procedures. The rest of the cDNAs unique to the FLJ collection (5,481) contained no obvious open reading frames (ORFs) and thus are candidate noncoding RNAs. About one-fourth of them (1,378) showed a clear pattern of splicing. The distribution of GC content of noncoding cDNAs was narrow and had a peak at approximately 42%, relatively low compared with that of protein-coding cDNAs.\n",
      "Here we present a finished sequence of human chromosome 15, together with a high-quality gene catalogue. As chromosome 15 is one of seven human chromosomes with a high rate of segmental duplication, we have carried out a detailed analysis of the duplication structure of the chromosome. Segmental duplications in chromosome 15 are largely clustered in two regions, on proximal and distal 15q; the proximal region is notable because recombination among the segmental duplications can result in deletions causing Prader-Willi and Angelman syndromes. Sequence analysis shows that the proximal and distal regions of 15q share extensive ancient similarity. Using a simple approach, we have been able to reconstruct many of the events by which the current duplication structure arose. We find that most of the intrachromosomal duplications seem to share a common ancestry. Finally, we demonstrate that some remaining gaps in the genome sequence are probably due to structural polymorphisms between haplotypes; this may explain a significant fraction of the gaps remaining in the human genome.\n",
      "The National Institutes of Health's Mammalian Gene Collection (MGC) project was designed to generate and sequence a publicly accessible cDNA resource containing a complete open reading frame (ORF) for every human and mouse gene. The project initially used a random strategy to select clones from a large number of cDNA libraries from diverse tissues. Candidate clones were chosen based on 5'-EST sequences, and then fully sequenced to high accuracy and analyzed by algorithms developed for this project. Currently, more than 11,000 human and 10,000 mouse genes are represented in MGC by at least one clone with a full ORF. The random selection approach is now reaching a saturation point, and a transition to protocols targeted at the missing transcripts is now required to complete the mouse and human collections. Comparison of the sequence of the MGC clones to reference genome sequences reveals that most cDNA clones are of very high sequence quality, although it is likely that some cDNAs may carry missense variants as a consequence of experimental artifact, such as PCR, cloning, or reverse transcriptase errors. Recently, a rat cDNA component was added to the project, and ongoing frog (Xenopus) and zebrafish (Danio) cDNA projects were expanded to take advantage of the high-throughput MGC pipeline.\n",
      "Histone deacetylase inhibitors (HDIs) induced expression of the B-ind1 protein that is a component of Rac-1-signaling pathways leading to the modulation of gene expression. In the present study, we have determined the structure of the human B-ind1 gene promoter region. The oligocapping method revealed that the transcriptional start site of the human B-ind1 gene is located at 166 bases upstream of the first adenine residue of the translation start site that is highly homologous to an initiator (Inr) consensus sequence. In reporter assays, transactivation of the B-ind1 promoter was observed up to 300 bp of the initiation site. Deletion analysis of the promoter region revealed that histone deacetylase inhibitors (HDIs)-induced luciferase response was regulated by the core promoter elements. Mutation introduced into the proximal CG-boxes decreased most of the basal and HDIs-induced promoter activity. These results suggested a novel mechanism, which implicate minimal core promoter elements as potential mediator of HDIs.\n",
      "BACKGROUND: SSeCKS is a major protein kinase C substrate with kinase scaffolding and metastasis-suppressor activity whose expression is severely downregulated in Src- and Ras-transformed fibroblast and epithelial cells and in human prostate, breast, and gastric cancers. We previously used NIH3T3 cells with tetracycline-regulated SSeCKS expression plus a temperature-sensitive v-Src allele to show that SSeCKS re-expression inhibited parameters of v-Src-induced oncogenic growth without attenuating in vivo Src kinase activity.\n",
      "METHODS: We use cDNA microarrays and semi-quantitative RT-PCR analysis to identify changes in gene expression correlating with i) SSeCKS expression in the absence of v-Src activity, ii) activation of v-Src activity alone, and iii) SSeCKS re-expression in the presence of active v-Src.\n",
      "RESULTS: SSeCKS re-expression resulted in the attenuation of critical Src-induced proliferative and pro-angiogenic gene expression including Afp, Hif-1alpha, Cdc20a and Pdgfr-beta, and conversely, SSeCKS induced several cell cycle regulatory genes such as Ptpn11, Gadd45a, Ptplad1, Cdkn2d (p19), and Rbbp7.\n",
      "CONCLUSION: Our data provide further evidence that SSeCKS can suppress Src-induced oncogenesis by modulating gene expression downstream of Src kinase activity.\n",
      "Cell signaling mechanisms often transmit information via posttranslational protein modifications, most importantly reversible protein phosphorylation. Here we develop and apply a general mass spectrometric technology for identification and quantitation of phosphorylation sites as a function of stimulus, time, and subcellular location. We have detected 6,600 phosphorylation sites on 2,244 proteins and have determined their temporal dynamics after stimulating HeLa cells with epidermal growth factor (EGF) and recorded them in the Phosida database. Fourteen percent of phosphorylation sites are modulated at least 2-fold by EGF, and these were classified by their temporal profiles. Surprisingly, a majority of proteins contain multiple phosphorylation sites showing different kinetics, suggesting that they serve as platforms for integrating signals. In addition to protein kinase cascades, the targets of reversible phosphorylation include ubiquitin ligases, guanine nucleotide exchange factors, and at least 46 different transcriptional regulators. The dynamic phosphoproteome provides a missing link in a global, integrative view of cellular regulation.\n",
      "Very long-chain fatty acids are produced through a four-step cycle. However, the 3-hydroxyacyl-CoA dehydratase catalyzing the third step in mammals has remained unidentified. Mammals have four candidates, HACD1-4, based on sequence similarities to the recently identified yeast Phs1, although HACD3 and HACD4 share relatively weak similarity. We demonstrate that all four of these human proteins are indeed 3-hydroxyacyl-CoA dehydratases, in growth suppression experiments using a PHS1-shut off yeast strain and/or in vitro 3-hydroxypalmitoyl-CoA dehydratase assays. HACD proteins exhibit distinct tissue-expression patterns. We also establish that HACD proteins interact with the condensation enzymes ELOVL1-7, with some preferences.\n",
      "Immobilized metal affinity chromatography (IMAC) is a common strategy used for the enrichment of phosphopeptides from digested protein mixtures. However, this strategy by itself is inefficient when analyzing complex protein mixtures. Here, we assess the effectiveness of using protein-based IMAC as a pre-enrichment step prior to peptide-based IMAC. Ultimately, we couple the two IMAC-based enrichments and MudPIT in a quantitative phosphoproteomic analysis of the epidermal growth factor pathway in mammalian cells identifying 4470 unique phosphopeptides containing 4729 phosphorylation sites.\n",
      "Protein kinases are pivotal regulators of cell signaling that modulate each other's functions and activities through site-specific phosphorylation events. These key regulatory modifications have not been studied comprehensively, because low cellular abundance of kinases has resulted in their underrepresentation in previous phosphoproteome studies. Here, we combine kinase-selective affinity purification with quantitative mass spectrometry to analyze the cell-cycle regulation of protein kinases. This proteomics approach enabled us to quantify 219 protein kinases from S and M phase-arrested human cancer cells. We identified more than 1000 phosphorylation sites on protein kinases. Intriguingly, half of all kinase phosphopeptides were upregulated in mitosis. Our data reveal numerous unknown M phase-induced phosphorylation sites on kinases with established mitotic functions. We also find potential phosphorylation networks involving many protein kinases not previously implicated in mitotic progression. These results provide a vastly extended knowledge base for functional studies on kinases and their regulation through site-specific phosphorylation.\n",
      "The eukaryotic cell division cycle is characterized by a sequence of orderly and highly regulated events resulting in the duplication and separation of all cellular material into two newly formed daughter cells. Protein phosphorylation by cyclin-dependent kinases (CDKs) drives this cycle. To gain further insight into how phosphorylation regulates the cell cycle, we sought to identify proteins whose phosphorylation is cell cycle regulated. Using stable isotope labeling along with a two-step strategy for phosphopeptide enrichment and high mass accuracy mass spectrometry, we examined protein phosphorylation in a human cell line arrested in the G(1) and mitotic phases of the cell cycle. We report the identification of >14,000 different phosphorylation events, more than half of which, to our knowledge, have not been described in the literature, along with relative quantitative data for the majority of these sites. We observed >1,000 proteins with increased phosphorylation in mitosis including many known cell cycle regulators. The majority of sites on regulated phosphopeptides lie in [S/T]P motifs, the minimum required sequence for CDKs, suggesting that many of the proteins may be CDK substrates. Analysis of non-proline site-containing phosphopeptides identified two unique motifs that suggest there are at least two undiscovered mitotic kinases.\n",
      "The mixture of phosphopeptides enriched from proteome samples are very complex. To reduce the complexity it is necessary to fractionate the phosphopeptides. However, conventional enrichment methods typically only enrich phosphopeptides but not fractionate phosphopeptides. In this study, the application of strong anion exchange (SAX) chromatography for enrichment and fractionation of phosphopeptides was presented. It was found that phosphopeptides were highly enriched by SAX and majority of unmodified peptides did not bind onto SAX. Compared with Fe(3+) immobilized metal affinity chromatography (Fe(3+)-IMAC), almost double phosphopeptides were identified from the same sample when only one fraction was generated by SAX. SAX and Fe(3+)-IMAC showed the complementarity in enrichment and identification of phosphopeptides. It was also demonstrated that SAX have the ability to fractionate phosphopeptides under gradient elution based on their different interaction with SAX adsorbent. SAX was further applied to enrich and fractionate phosphopeptides in tryptic digest of proteins extracted from human liver tissue adjacent to tumorous region for phosphoproteome profiling. This resulted in the highly confident identification of 274 phosphorylation sites from 305 unique phosphopeptides corresponding to 168 proteins at false discovery rate (FDR) of 0.96%.\n",
      "INTERACTIONS:\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"Q9P035\", \"intActId\": \"EBI-359013\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"P13569\", \"geneName\": \"CFTR\", \"intActId\": \"EBI-349854\"}, \"numberOfExperiments\": 15, \"organismDiffer\": false}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"Q9P035\", \"intActId\": \"EBI-359013\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q03463\", \"chainId\": \"PRO_0000278740\", \"intActId\": \"EBI-8803426\"}, \"numberOfExperiments\": 2, \"organismDiffer\": true}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"Q9P035\", \"intActId\": \"EBI-359013\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q99IB8\", \"chainId\": \"PRO_0000045602\", \"intActId\": \"EBI-6927873\"}, \"numberOfExperiments\": 2, \"organismDiffer\": true}\n",
      "{\"interactantOne\": {\"uniProtKBAccession\": \"Q9P035\", \"intActId\": \"EBI-359013\"}, \"interactantTwo\": {\"uniProtKBAccession\": \"Q9WMX2\", \"chainId\": \"PRO_0000037551\", \"intActId\": \"EBI-6863748\"}, \"numberOfExperiments\": 3, \"organismDiffer\": true}\n",
      "PATHWAYS:\n",
      "\n",
      "<END_CONTEXT>\n"
     ]
    }
   ],
   "source": [
    "print(full_context)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "with open(\"example_context_large.txt\", \"w\") as f:\n",
    "    f.write(full_context)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "genai",
   "language": "python",
   "name": "genai"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
